import React, { useState, useEffect } from "react";
import Axios from "axios";
import Article from "./Article";
import Header from "./Header";
import Collapsible from "react-collapsible";

function SearchMain() {
  const [query, setQuery] = useState("");
  const [articles, setArticles] = useState([]);
  const [alert, setAlert] = useState("");

  //url for search function
  const url = `http://localhost:8000/answer/?query=${query}`;

  const getData = async () => {
    if (query !== "") {
      const result = await Axios.get(url);
      if (!result.data) {
        return setAlert("No article retrieved for this query.");
      }
      console.log(result);
      setArticles(result.data);
      setQuery("");
      setAlert("");
    } else {
      setAlert("Please fill the form");
    }
  };

  const onChange = e => setQuery(e.target.value);

  const onSubmit = e => {
    e.preventDefault();
    getData();
  };

  return (
    <div className="App-header">
      <Header />
      {/*Search Input*/}
      <form onSubmit={onSubmit} className="search-form" htmlFor="search-input">
        <input
          type="text"
          name="query"
          value={query}
          id="search-input"
          placeholder="Search..."
          autoComplete="off"
          onChange={onChange}
        />
        <input type="submit" value="Search" />
      </form>

      <ul className="articles">
        <div className="query_display">
          <h3>Query:&nbsp;{query}</h3>
        </div>
        {articleSample !== [] &&
          articleSample.map(article => (
            <Article key={article.paper_id} article={article} />
          ))}
      </ul>
    </div>
  );
}

export default SearchMain;

const articleSample = [
  {
    answer: {
      score: "75.31819711516434",
      sents: [
        "From the insight of glucose metabolism disorder: Oxygen therapy and blood glucose monitoring are crucial for quarantined COVID-19 patients",
        "Majority of the COVID-19 patients in China have been cured following the ever-improving treatment methods, and oxygen supplement has been applied as a common treatment strategy for cases in hospitals and cabin hospitals (China National Health Commission).",
        "Since 81% of the confirmed cases are in mild condition with non-or mild pneumonia, most of COVID-19 patients outside China are now recommended for self-cured under home quarantine.",
        "However, it may only take 9 days for the mild COVID-19 patients without effective nursing to develop into serious or critical cases with acute respiratory distress syndrome (ARDS) (Guan et al., 2020; Huang et al., 2020) .",
        "Therefore, it is critical to promote the self-cure rate of the mild cases and prevent the disease from deterioration.",
        "Considering the harmful consequences induced by hypoxemia and advantages of oxygen therapy in controlling glucose metabolism (Handy et al., 2005) , we suggest that adequate oxygen intake is also essential for the newly confirmed or suspected cases under home quarantine."
      ]
    },
    paper_id: "PMC7126624",
    doi: "10.1016/j.bios.2006.04.024",
    title:
      "DNA hybridization sensor based on aurothiomalate electroactive label on glassy carbon electrodes",
    doc_date: "2007-01-15",
    authors: [
      "Alfredo.de la Escosura-Muñiz",
      "María.González-García",
      "Agustín.Costa-García"
    ],
    summary: "",
    abstract: {
      text: [""],
      tags: {
        sciwing: []
      }
    },
    bodyText: {
      section_header: {
        original: [
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Apparatus and electrodes ::: Experimental",
          "Apparatus and electrodes ::: Experimental",
          "Apparatus and electrodes ::: Experimental",
          "Apparatus and electrodes ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Labeling of the target strand with sodium aurothiomalate ::: Methods ::: Experimental",
          "Labeling of the target strand with sodium aurothiomalate ::: Methods ::: Experimental",
          "Electrode pre-treatment ::: Methods ::: Experimental",
          "Immobilization of the probe ::: Methods ::: Experimental",
          "Hybridization reaction ::: Methods ::: Experimental",
          "Hybridization reaction ::: Methods ::: Experimental",
          "Recording the analytical signal: catalytic effect of gold on the silver electrodeposition ::: Methods ::: Experimental",
          "Recording the analytical signal: catalytic effect of gold on the silver electrodeposition ::: Methods ::: Experimental",
          "Recording the analytical signal: catalytic effect of gold on the silver electrodeposition ::: Methods ::: Experimental",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Hybridization reaction ::: Results and discussion",
          "Hybridization reaction ::: Results and discussion",
          "Hybridization reaction ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Conclusions",
          "Conclusions",
          "Conclusions",
          "Conclusions",
          "Conclusions",
          "Conclusions",
          "Conclusions"
        ],
        generic: [
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "conclusions",
          "conclusions",
          "conclusions",
          "conclusions",
          "conclusions",
          "conclusions",
          "conclusions"
        ]
      },
      text: [
        "DNA diagnostic has become an important area of molecular biology and biotechnology studies. The detection of specific base sequences in human, viral and bacterial nucleic acid is becoming increasingly important in several areas, with applications ranging from the detection of disease-causing and food-contaminating organisms to forensic and environmental research.",
        "Conventional methods for the analysis of specific gene sequences are based on either direct sequencing or DNA hybridization methods. Although the DNA sequencing has undergone very big progress in the last decade (Kling, 2003, Kartalov and Quake, 2004, Aborn et al., 2005), these traditional methods, based on the coupling of electrophoretic separations and radioisotopic (32P) detection, are generally labour intensive and time consuming. Therefore, the second option is more commonly used in diagnostic laboratories.",
        "Genosensors (or DNA hybridization biosensors) offer a promising alternative to carry out the specific gene sequence identification (De los Santos-Álvarez et al., 2004, Wang, 2002, Paleček et al., 1998). These biosensors commonly rely on the immobilization of a single-stranded (ss) oligonucleotide probe onto a transducer surface to recognize – by hybridization – its complementary target sequence.",
        "Among the different kind of transducers found in the literature – chemiluminescent (Nguyen and Heffelfinger, 1995), quartz crystal microbalance (Hashimoto et al., 1994, Steel et al., 1998), fiber optical (Piunno et al., 1995), evanescent wave (Watts et al., 1995) or an-acoustic wave (Su et al., 1996) – those based on electrochemical transduction, due to their high sensitivity, small dimensions, low cost and compatibility with micro-fabrication technology provide a very interesting alternative.",
        "These electrochemical biosensing devices can monitor sequence-specific hybridization processes: (i) directly, based on the intrinsic DNA electroactivity (Kerman et al., 2005, Erdem et al., 2004, Ozkan et al., 2002, Lucarelli et al., 2002) or (ii) indirectly by measuring changes in the electrical properties of the electrode–solution interface – electrochemical properties of conducting polymers (Thompson et al., 2003, Cha et al., 2003, Wang et al., 1999), capacitance (Berney et al., 2000) or impedance (Souteyrand et al., 1997, Alfonta et al., 2001) – or using different covalently and non-covalently bound markers.",
        "Non-covalent markers (hybridization indicators) are electroactive compounds – metal complexes (Wang et al., 1997, Zhao and Ju, 2004, Nojima et al., 2003, Kara et al., 2002, Aoki and Umezawa, 2003) or organic compounds (Zhu et al., 2005, Teh et al., 2005, Marrazza et al., 1999, Hashimoto et al., 1998) – that interact in a different way with single strands (ss-) and double strands (ds-) of DNA.",
        "Covalently bound markers can be subdivided in two groups, electroactive and non-electroactive. Ferrocene and its derivatives (Nakayama et al., 2002, Xu et al., 2000, Xu et al., 2001), Pt(II) complex (Hernández-Santos et al., 2005), colloidal gold (Ozsoz et al., 2003, Cai et al., 2002), or tris(2,2′-bipyridyl)cobalt(III)-doped silica nanoparticles (Zhu et al., 2003) are used as electroactive markers, while as non-electroactive marker, the use of enzymes, linked directly to the DNA strand (Zhang et al., 2003, Zhang et al., 2004) or indirectly through biotin–avidin (Abad-Valle et al., 2005, Azek et al., 2000) or fluorescein–antifluorescein bridges (Hernández-Santos et al., 2004), has been extensively reported.",
        "As can be seen, metal complexes and metal nanoparticles have an important role in the development of electrochemical genosensors. In addition to the works cited above, where metal complexes are used as hybridization indicators or electroactive labels, examples of transition metal complexes, e.g. Ru(bpy)3\n2+, Os(bpy)3\n2+, mediating the electro-oxidation of nucleobases, have also been found in literature (Yang et al., 2002, Gore et al., 2003).",
        "In this work, it was used for first time sodium aurothiomalate as electroactive label in a DNA hybridization biosensor. This gold(I) complex has been previously used in our laboratory with success as electroactive label for immunosensor devices using glassy carbon electrodes (GCEs) as electrochemical transducers (de la Escosura-Muñiz et al., 2004a). The electrochemical detection in these immunosensors is based on the catalytic effect of ionic gold on silver electrodeposition (de la Escosura-Muñiz et al., 2004b).",
        "GCEs have been widely used in the construction of electrochemical DNA hybridization biosensors in combination with different oligonucleotides immobilization methods, such as (i) film entrapment using polymers (Wang et al., 1999, Xu et al., 2001, Cai et al., 2002, Zhu et al., 2003), (ii) covalent attachment on functionalized glassy carbon electrodes—polymers, carbon nanotubes, etc. (Zhao and Ju, 2004, Kara et al., 2002, Zhu et al., 2005, Teh et al., 2005, Piro et al., 2005, Cai et al., 2003) or (iii) by electrostatic adsorption on gold colloid modified surfaces (Lin et al., 2002).",
        "Direct adsorption of oligonucleotides onto pre-oxidized glassy carbon electrodes is used in this work, in combination with the electroactive label sodium aurothiomalate for the electrochemical detection of a SARS virus sequence, using a silver catalytic electrodeposition method.",
        "Cyclic voltammetric experiments were performed with an ECO CHEMIE Autolab PGSTAT 10 potentiostat interfaced to a Pentium 120 computer system and controlled by an Autolab GPES Version 4.6 for Windows 98.",
        "All measurements were carried out at room temperature in a 20 mL cell (protected from light) with a three-electrode configuration. Working glassy carbon electrodes (GCEs) were home made, using 3 mm diameter glassy carbon rods (Goodfellow, Spain), sealed into Teflon holders with a spurr resin for electron microscopy purchased from Sigma (Spain). Electrical contact was a brass rod welded to glassy carbon with a silver loaded conductive epoxy resin purchased from Circuit Works (USA). The renewal of the glassy carbon surface was achieved by polishing with 1.0 and 0.3 μm alpha-alumina on a microcloth polishing sheet of 8 in, followed by washing in an ultrasonic Selecta bath for 5 min.",
        "A platinum wire as counter electrode and an Ag/AgCl reference electrode were used.",
        "A Metrohm AG Herisau magnetic stirrer was used for the electrochemical pre-treatment of the electrode surface and the gold electrodeposition and oxidation.",
        "Synthetic 30-mer oligonucleotides were obtained from Eurogentec (Spain). The target sequence employed corresponds to a portion of the severe acute respiratory syndrome (SARS) virus, precisely the bases comprised between 29218 and 29247, both included. For selectivity studies, single-base and three-base-mismatch strands were also purchased. Mismatches are located in bases number 12 and 5, 15 and 26, respectively.Target (50 nmol): 5′-ACA-GAG-CCT-AAA-AAG-GAC-AAA-AAG-AAA-AAG-3′;Thiolated target (50 nmol): 5′-ACA-GAG-CCT-AAA-AAG-GAC-AAA-AAG-AAA-AAG-SH-3′;Single-base-mismatch target (49 nmol): 5′-ACA-GAG-CCT-AAC-AAG-GAC-AAA-AAG-AAA-AAG-3′;Three-base-mismatch target (184 nmol): 5′-ACA-GCG-CCT-AAA-AAC-GAC-AAA-AAG-AGA-AAG-3′;The probe is a complementary strand of the target that was also obtained from Eurogentec. Probe (50 nmol): 5′-CTT-TTT-CTT-TTT-GTC-CTT-TTT-AGG-CTC-TGT-3′.\n",
        "Oligonucleotide solutions were prepared in TE buffer, pH 8 (10 mM Tris–HCl buffer solution, 1 mM in EDTA). Aliquots were prepared and maintained at −20 °C.",
        "Working solutions of the oligonucleotide probe were made in 0.1 M Tris, pH 7.2 buffer, while all gold labeled oligonucleotide target strands were diluted in a 2 × SSC buffer (300 mM sodium chloride/30 mM sodium citrate), pH 7.2. These solutions were conserved at 4 °C. For selectivity studies, oligonucleotide strands were diluted in 2 × SCC buffer or in this buffer containing 50% of formamide. For the labeling procedure, target strands were diluted in an aqueous solution of 0.15 M NaCl with pH adjusted to 7.5 with 0.1 M NaOH.",
        "Sodium aurothiomalate was obtained from Aldrich (Spain). It was reconstituted in 0.15 M NaCl solution and protected from light. Dilutions of this stock solution were prepared daily in an unbuffered aqueous solution of 0.15 M NaCl, with pH adjusted to 7.5 with 0.1 M NaOH.",
        "Tris(hydroxymethyl)aminomethane (Tris) and formamide were purchased from Sigma (Spain). EDTA was obtained from Fluka (Spain).",
        "Silver nitrate was obtained from Sigma (Spain). Solutions were prepared using ultra-pure water and were protected from light.",
        "Analytical grade (Merck, Spain) NaCl, HCl, H2SO4, NH3, KCN, NaOH and sodium citrate were used. Their solutions were prepared with Millipore Milli-Q system, excepting KCN solutions, which were prepared with a 0.1 M NaOH solution.",
        "Ultra-pure water obtained with a Milli-Q plus 185 from Millipore Ibérica S.A. (Spain) was used for all solutions.",
        "Alumina powders of 1.0 and 0.3 μm grain sizes and microcloth polishing sheet of 8 in were purchased from Buehler (Germany).",
        "Dialysis procedures were carried out with Slide-A-Lizer dialysis cassettes, 3500 MWCO from Pierce (USA).",
        "The conjugation of aurothiomalate to single strands was carried out according to the procedure described in a previous work to label immunoglobulins (de la Escosura-Muñiz et al., 2004a). An aliquot of 50 μL of a 200 ng μL−1 target strand solution was mixed with an aliquot of 450 μL of a 390 ng μL−1 sodium aurothiomalate solution. The reaction was carried out at 37 °C for 24 h. After that, the conjugate (strand-Au) was purified by dialysis against 200 mL of 0.15 M NaCl, pH 7.5, unbuffered solution, for 48 h at room temperature.",
        "Thiolated and non-thiolated targets, single-base and three-base-mismatch complementary strands and a non-complementary strand were labeled following this procedure.",
        "The smoothed glassy carbon surfaces were pre-treated before each assay by an electrochemical pre-treatment in 0.1 M HCl. Working electrodes were immersed in a 0.1 M HCl stirred solution and a potential of +1.40 V was applied for 2 min. After that, the electrode surfaces were washed with 0.1 M Tris–HCl, pH 7.2, buffer.",
        "The probe immobilization was performed by physical adsorption, onto the inverted electrode, at room temperature. Five GCEs were used simultaneously and the adsorption was carried out depositing a 50 μL droplet of a 1 ng μL−1 oligonucleotide probe solution on each electrode surface, and left there for 60 min. Then, a washing step was performed with the hybridization buffer solution.",
        "After the probe immobilization and the washing step, the hybridization reaction took place for 30 min with a 30 μL droplet of the complementary gold labeled strand, 2 × SCC buffer pH 7.2 diluted, at room temperature. After that, the electrode surfaces were washed with ultra-pure water.",
        "The same procedure was performed to obtain the background signals, using a gold labeled non-complementary strand, and for selectivity studies with gold labeled single-base and three-base-mismatch complementary strands. For the selectivity studies, 2 × SCC buffer, pH 7.2, and this buffer containing 50% of formamide were used.",
        "The electrodes were immersed in a stirred 0.1 M HCl solution, and the gold, which was bound to the target strand, was electrodeposited on the electrode surfaces by applying a potential of −1.00 V for 5 min. After a washing step with ultra-pure water, the electrodes were immersed in a stirred 0.1 M H2SO4 solution and an oxidation step was carried out applying a potential of +1.40 V for 60 s. After that, the electrodes were rinsed with ultra-pure water and introduced in an unstirred 1.0 M NH3 solution containing silver nitrate at a fixed concentration (2.0 × 10−4\n M) and held at a deposition potential of −0.14 V for 60 s. Then, cyclic voltammograms were scanned from deposition potential to +0.30 V at a scan rate of 50 mV s−1, obtaining the analytical signal. Finally, in order to remove the gold from the electrode surfaces, the GCEs were immersed after each measurement in another cell containing a 0.1 M KCN stirred solution for 2 min in open circuit.",
        "This analytical procedure, based on the catalytic effect of electrodeposited gold on the electroreduction of silver ions (de la Escosura-Muñiz et al., 2004b) has been previously optimized, using bovine serum albumin (BSA) and a rabbit immunoglobulin G (RIgG) labeled with aurothiomalate (de la Escosura-Muñiz et al., 2004a). Briefly, this detection protocol involves a reduction step at −1.00 V in HCl 0.1 M for the gold label electrodeposition on the electrode surface, followed by an oxidation step in 0.1 M H2SO4 necessary to remove the hydrogen generated in the gold deposition step and to oxidize this gold. Then, by selecting an adequate potential (−0.14 V), the silver contained in the solution of AgNO3 2 × 10−4\n M–NH3 1.0 M is reduced to metallic silver only in presence of gold deposited on the electrode surface. Finally, an anodic scan is performed and the reduced silver is oxidized at +0.08 V, which constitutes the analytical signal.",
        "As the amount of silver electrodeposited at this potential is proportional to the gold deposited on the electrode surface, the silver stripping allows the determination of the gold labeled target that has hybridized with the probe. When a non-complementary gold labeled strand is assayed no hybridization takes place, so no gold is deposited on the electrode surface. Thus, no silver is reduced at the chosen deposition potential, and no signal is obtained.",
        "It is well known that sodium aurothiomalate binds to the albumin molecules mainly through the thiol group of the cysteine 34, by exchanging a proton (Moller-Pedersen, 1987). Based on this interaction between gold and thiol, our group has successfully used sodium autothiomalate for labeling different biological molecules, such as BSA and immunoglobulins, in order to design electrochemical metal immunoassays and immunosensors (de la Escosura-Muñiz et al., 2004a).",
        "Thus, this high affinity between gold and thiol group was applied in order to label a thiolated oligonucleotide. The product of the dialysis step was then evaluated by direct adsorption on the electrode surface, previously pre-treated at +1.40 V (versus Ag/AgCl) for 2 min in 0.1 M HCl stirred solution. A 50 μL droplet of a 1 ng μL−1 solution of the thiolated oligonucleotide, after the labeling process, was deposited on the electrode surface and left there for 30 min, before recording the analytical signal.",
        "To verify the gold(I)–oligonucleotide binding and to know if this interaction is through the thiol group, a solution of sodium aurothiomalate 390 ng μL−1 that was not mixed with any oligonucleotide, and a non-thiolated oligonucleotide with identical sequence, were subjected to the same procedure (reaction + dialysis + adsorption).",
        "The results obtained are showed in Fig. 2\n. As it can be seen in this figure, very similar analytical signals were obtained for both, the thiolated (Fig. 2a) and the non-thiolated oligonucleotide (Fig. 2b), while, as it could be expected, no analytical signal was obtained for the aurothiomalate solution assayed (Fig. 2c), because, as it is known by our previous works, sodium aurothiomalate is totally dialysed and anyway, it is not adsorbed on the electrode surface under these experimental conditions. The reproducibility of the signals showed in Fig. 2a and b was checked, using five different GCEs, obtaining a R.S.D. of 8% (n\n = 5, Ip average = 6.6 μA) and 7% (n\n = 5, Ip average = 6.4 μA), respectively.",
        "These results indicate that oligonucleotide is adsorbed on the GCE surface and it has been successfully labeled with sodium aurothiomalate.",
        "The DNA binding properties of sodium aurothiomalate and other gold(I) complexes have been studied by Blank and Dabrowiak (1984) using absorption and circular dichroism spectroscopies. This study concludes that sodium aurothiomalate does not bind to calf thymus DNA, while others gold(I) complexes which have an easily displaced ligand, e.g. Cl− or Br− are capable of interacting in a non-denaturing fashion with the guanine and cytosine residues of calf thymus DNA.",
        "To the best of our knowledge, this is the unique study of gold(I)–DNA complexes that includes sodium aurothiomalate. So, we only have evidence of its not interaction with calf thymus DNA, but we have no information about its binding to ssDNA or to DNA nucleosides.",
        "Therefore, the aurothiomalate–oligonucleotide binding mechanism is not clear, since not only the thiolated strand was labeled, being the interaction not through the thiol group, at least in its totality, as it could be expect. Besides, this fact allows us to label unmodified oligonucleotides which is economically advantageous.",
        "Physical adsorption is used in order to immobilize the oligonucleotide on the pre-oxidized GCE surface at room temperature. The influence of the probe strand concentration and adsorption time on the analytical signal were evaluated following the analytical procedure previously described, using in both cases 30 min of hybridization and 0.1 ng μL−1 gold labeled target solution.",
        "\nFig. 3\nshows the influence of the probe strand concentration on the analytical signal, using an adsorption time of 60 min. The peak current increases with the probe concentration up to 1 ng μL−1, where it reaches a plateau. So, 1 ng μL−1 was chosen for further studies.",
        "A very important fact obtained from this optimization assay is that non-specific adsorptions of the gold labeled target on the electrode surface are not observed for the concentration assayed (0.1 ng μL−1), since when no probe is immobilized, no analytical signal is recorded. The absence of non-specific adsorptions is a very important fact in the development of DNA hybridization sensors, since a blocking step is not needed, which simplifies the procedure, and the sensitivity of the assay is notably improved.",
        "Two factors could be responsible for this fact. One of them is that the gold labeled target concentration is not higher enough to adsorb on the electrode surface in 30 min. The other one is that, although the strand is adsorbed, the low amount of gold deposited on the electrode does not catalyze the silver reduction.",
        "Once the probe concentration was optimized, the effect of the probe adsorption time (from 5 to 90 min) on the analytical signal was evaluated. The obtained results (data not shown) show that the peak current increases upon raising the adsorption time up to 60 min. Beyond this point, the electrode response changes slightly with the adsorption time. Thus, an adsorption time of 60 min is chosen for further studies. Besides of the peak current increase observed with the adsorption time, a better reproducibility of the analytical signal is also obtained when the adsorption time is increased.",
        "Electrostatic adsorption of the oligonucleotide probe was also tested using different times (1–10 min) and potentials (+0.3 to +1.0 V) without success.",
        "After the optimization of the genosensor sensing phase, the hybridization reaction time (between 5 and 60 min) was studied using a 30 μL droplet of a 0.1 ng μL−1 solution of gold labeled target strand in 2 × SCC, pH 7.2, buffer solution. The analytical signal recorded increases with the reaction time, reaching a plateau for a hybridization time of 30 min (data not shown). So, a hybridization time of 30 min is chosen for further studies.",
        "The background signal was also evaluated using a 0.1 ng μL−1 solution of the gold labeled non-complementary strand. When this strand is assayed, no hybridization reaction is expected, so this gold labeled strand will not be captured by the probe immobilized on the electrode surface. As the analytical signal is based on the catalytic effect of electrodeposited gold on the silver electroreduction, when no gold is deposited on the electrode surface, no silver is reduced at the chosen deposition potential, and no signal is obtained.",
        "As can be seen in Fig. 4\n, no hybridization reaction takes place working with the gold labeled non-complementary strand, and a perfect discrimination between complementary (Fig. 4a) and non-complementary (Fig. 4b) strand is obtained under the experimental conditions optimized.",
        "Following the analytical procedure previously described, the higher the concentration of labeled target in solution, the higher is the amount of gold that is electrodeposited, and subsequently, an increase in the analytical signal is obtained.",
        "The peak current showed a good linear relationship with the concentration of gold labeled target prepared in 2 × SCC, pH 7.2, buffer solution, in the range from 10 to 200 pg μL−1, with a correlation coefficient of 0.9984, according to the following equation:Ip (μA)=0.049[target] (pg μL−1)+0.198 (n=5).\n",
        "The limit of detection (calculated as the concentration corresponding to three times the standard deviation of the estimate) of the target strand was 5.0 pg μL−1 (15 fmol in 30 μL).",
        "Background signals, using the gold labeled non-complementary strand, and non-specific adsorption of the gold labeled target were also evaluated. As it was expected in both cases, no analytical signals were obtained working with concentrations between 10 and 200 pg μL−1.",
        "The reproducibility of the analytical signal was checked. Using five different GCEs, a R.S.D. of 10% (n\n = 5, Ip average = 2.1 μA) was obtained for a 50 pg μL−1 solution of the complementary strand.",
        "Finally, it was also evaluated the selectivity of the DNA hybridization sensor, using different concentrations (10, 25, 50, 100 and 200 pg μL−1) of both single-base and three-base-mismatch gold labeled complementary strands prepared in 2 × SCC, pH 7.2, buffer solution. As it is shown in Fig. 5\n, the sensor allows us to discriminate between the complementary strand and the three-base-mismatch complementary strand. However, with the single-base mismatch complementary strand, the analytical signals recorded are similar than those obtained for the complementary strand.",
        "In order to discriminate a single-base-mismatch, other saline concentrations of the hybridization buffer were tested, but the discrimination between single-base-mismatch complementary strand and the target strand was not improved.",
        "More stringent experimental conditions were also tested, adding different concentrations of formamide to the 2 × SCC, pH 7.2, hybridization buffer. It is well known that this molecule makes the hybridization reaction more difficult by decreasing the melting point of DNA.",
        "A concentration of 50% of formamide was necessary to discriminate between single-base-mismatch complementary strand and the target strand. The results obtained for 10, 25, 50, 100 and 200 pg μL−1 strand concentrations are shown in Fig. 6\n. Working with this formamide concentration, the analytical signals obtained for the target strand decrease, while the analytical signals obtained for the three-base-mismatch complementary strand were approaching zero.",
        "Under these more stringent conditions, a new calibration curve was obtained. A linear relationship between peak current and concentration of oligonucleotide target was obtained for concentrations between 10 and 200 pg μL−1, with a correlation coefficient of 0.997, according to the following equation:Ip (μA)=0.016[target] (pg μL−1)+0.0925 (n=5).\n",
        "The limit of detection calculated as described above, was 10 pg μL−1 (30 fmol in 30 μL).",
        "As can be seen, the sensitivity (slope) of the assay decreases under these more stringent conditions, due to the minor peak current recorded for all concentrations of gold labeled target assayed. However, the same linear range from 10 to 200 pg μL−1 were obtained working with both non-stringent and stringent experimental conditions.",
        "The R.S.D. (50 pg μL−1 target strand) for five parallel experiments under these experimental conditions was 8.9% with a mean peak current of 0.84 μA.",
        "This work describes, for the first time, the use of sodium aurothiomalate as electroactive label in a DNA hybridization biosensor.",
        "About the interaction between this gold(I)complex and oligonucleotides, we only can conclude that the presence of a thiol group on the strand, which acts as a binding site for gold in other biological molecules, is not necessary to carry out an effective interaction.",
        "However, the hybridization of gold labeled target strand onto the surface of a probe modified GCE is performed successfully, demonstrating that DNA bases of the target are available for the hybridization after its labeling with the gold(I) complex.",
        "Although the sensitivity of the sensor is lower than that obtained with an enzymatic method on gold band electrodes (Abad-Valle et al., 2005), the analysis time is considerably shorter, since in the enzymatic method, a blocking step, a biotin–streptavidin interaction to link the enzyme to the duplex and an enzymatic reaction are needed before to obtain the analytical signal.",
        "Besides, the sensitivity of this genosensor is competitive with others reported that use Pt(II) complex – 20-mer oligonucleotide (Hernández-Santos et al., 2005) – or tris(2,2′-bipyridil)cobalt(III)-doped silica nanoparticle – 24-mer oligonucleotide (Zhu et al., 2003) – electroactive label; and with others gold nanoparticle-based hybridization assays – 19-mer oligonucleotide (Wang et al., 2001) – that use electrochemical detection.",
        "The reported genosensor is simple, economical and it is able to discriminate single-base mismatch on the target.",
        "Work is in progress trying to elucidate the binding mechanism between aurothiomalate and oligonucleotides and adapting the system to another carbon electrodes in order to improve the sensitivity."
      ],
      tags: {
        sciwingI2B2: {
          "0,10": "problem",
          "0,11": "problem",
          "0,12": "problem",
          "0,13": "problem",
          "0,14": "problem",
          "0,17": "problem",
          "0,24": "problem",
          "0,37": "problem",
          "0,40": "problem",
          "0,41": "problem",
          "0,43": "problem",
          "0,45": "problem",
          "0,46": "problem",
          "0,47": "problem",
          "0,48": "problem",
          "0,49": "problem",
          "0,51": "problem",
          "0,52": "problem",
          "0,53": "problem",
          "0,68": "problem",
          "0,71": "problem",
          "0,72": "problem",
          "0,77": "problem",
          "0,78": "problem",
          "0,87": "problem",
          "0,88": "problem",
          "0,90": "problem",
          "0,91": "problem",
          "0,107": "problem",
          "0,108": "problem",
          "0,109": "problem",
          "0,115": "problem",
          "0,116": "problem",
          "0,117": "problem",
          "0,121": "problem",
          "0,122": "problem",
          "0,123": "problem",
          "0,125": "problem",
          "0,128": "problem",
          "0,129": "problem",
          "1,0": "problem",
          "1,5": "problem",
          "1,6": "problem",
          "1,8": "problem",
          "1,9": "problem",
          "1,13": "problem",
          "1,14": "problem",
          "1,15": "problem",
          "1,16": "problem",
          "1,17": "problem",
          "1,18": "problem",
          "1,23": "problem",
          "1,24": "problem",
          "1,25": "problem",
          "1,28": "problem",
          "1,37": "problem",
          "1,45": "problem",
          "1,46": "problem",
          "1,51": "problem",
          "1,52": "problem",
          "1,54": "problem",
          "1,66": "treatment",
          "1,68": "treatment",
          "1,69": "treatment",
          "1,70": "treatment",
          "1,71": "treatment",
          "1,72": "treatment",
          "1,74": "treatment",
          "1,76": "treatment",
          "1,77": "treatment",
          "1,78": "treatment",
          "1,79": "treatment",
          "1,80": "treatment",
          "1,82": "treatment",
          "1,90": "problem",
          "1,91": "problem",
          "1,95": "problem",
          "1,96": "problem",
          "1,105": "problem",
          "1,106": "problem",
          "1,108": "test",
          "1,109": "test",
          "1,123": "problem",
          "1,124": "problem",
          "1,126": "problem",
          "1,127": "problem",
          "1,128": "problem",
          "1,131": "problem",
          "1,132": "problem",
          "1,133": "problem",
          "1,134": "problem",
          "1,135": "problem",
          "1,139": "problem",
          "1,140": "problem",
          "1,142": "problem",
          "1,143": "problem",
          "1,144": "problem",
          "1,146": "problem",
          "1,147": "problem",
          "1,149": "problem",
          "1,150": "problem",
          "1,151": "problem",
          "1,152": "problem",
          "1,157": "problem",
          "1,158": "problem",
          "1,159": "problem",
          "1,160": "problem",
          "1,162": "problem",
          "1,163": "problem",
          "1,165": "problem",
          "1,166": "problem",
          "1,167": "problem",
          "1,168": "problem",
          "1,173": "problem",
          "1,174": "problem",
          "1,175": "problem",
          "1,176": "problem",
          "1,184": "problem",
          "1,189": "problem",
          "1,190": "problem",
          "1,192": "problem",
          "1,193": "problem",
          "2,0": "problem",
          "2,1": "problem",
          "2,2": "problem",
          "2,4": "problem",
          "2,5": "problem",
          "2,6": "problem",
          "2,7": "problem",
          "2,31": "problem",
          "2,35": "test",
          "2,36": "test",
          "2,45": "problem",
          "2,70": "test",
          "2,71": "test",
          "2,73": "problem",
          "2,86": "problem",
          "2,87": "problem",
          "2,90": "test",
          "2,91": "test",
          "2,93": "problem",
          "2,94": "problem",
          "2,95": "problem",
          "2,96": "problem",
          "2,98": "problem",
          "2,99": "problem",
          "2,100": "problem",
          "2,101": "problem",
          "2,103": "problem",
          "2,105": "problem",
          "2,106": "problem",
          "2,107": "problem",
          "2,113": "problem",
          "2,114": "problem",
          "2,116": "problem",
          "2,117": "problem",
          "2,118": "problem",
          "2,121": "problem",
          "2,122": "problem",
          "2,127": "problem",
          "2,128": "problem",
          "2,136": "problem",
          "2,137": "problem",
          "2,138": "problem",
          "2,139": "problem",
          "2,145": "problem",
          "2,150": "problem",
          "2,155": "test",
          "2,156": "test",
          "2,167": "problem",
          "2,169": "problem",
          "2,170": "problem",
          "2,171": "problem",
          "2,172": "problem",
          "2,173": "problem",
          "2,174": "problem",
          "2,175": "problem",
          "2,176": "problem",
          "2,178": "problem",
          "2,179": "problem",
          "2,188": "problem",
          "2,189": "problem",
          "2,192": "problem",
          "2,193": "problem",
          "2,194": "problem",
          "2,195": "problem",
          "2,197": "problem",
          "2,199": "test",
          "2,200": "test",
          "2,201": "test",
          "2,202": "test",
          "2,209": "problem",
          "2,222": "problem",
          "2,223": "problem",
          "2,224": "problem",
          "2,229": "problem",
          "2,230": "problem",
          "2,236": "problem",
          "2,237": "problem",
          "3,3": "problem",
          "3,4": "problem",
          "3,5": "problem",
          "3,23": "problem",
          "3,24": "problem",
          "3,29": "problem",
          "3,30": "problem",
          "3,31": "problem",
          "3,37": "test",
          "3,38": "test",
          "3,48": "problem",
          "3,49": "problem",
          "3,50": "problem",
          "3,54": "problem",
          "3,55": "problem",
          "3,60": "problem",
          "3,61": "problem",
          "3,68": "problem",
          "3,69": "problem",
          "3,71": "problem",
          "3,72": "problem",
          "3,73": "problem",
          "3,77": "problem",
          "3,85": "problem",
          "3,86": "problem",
          "3,88": "problem",
          "3,89": "problem",
          "3,97": "problem",
          "3,98": "problem",
          "3,99": "problem",
          "3,100": "problem",
          "3,101": "problem",
          "3,102": "problem",
          "3,103": "problem",
          "3,104": "problem",
          "3,106": "problem",
          "3,116": "test",
          "3,117": "test",
          "3,122": "problem",
          "3,123": "problem",
          "3,127": "problem",
          "3,128": "problem",
          "3,133": "test",
          "3,141": "problem",
          "3,144": "problem",
          "3,148": "problem",
          "3,150": "problem",
          "3,151": "problem",
          "3,152": "problem",
          "3,153": "problem",
          "3,154": "problem",
          "3,156": "problem",
          "3,157": "problem",
          "4,0": "problem",
          "4,1": "problem",
          "4,2": "problem",
          "4,4": "problem",
          "4,5": "problem",
          "4,11": "problem",
          "4,13": "problem",
          "4,15": "problem",
          "4,16": "problem",
          "4,17": "problem",
          "4,23": "problem",
          "4,24": "problem",
          "4,28": "problem",
          "4,30": "problem",
          "4,31": "problem",
          "4,32": "problem",
          "4,33": "problem",
          "4,35": "problem",
          "4,36": "problem",
          "4,45": "problem",
          "4,46": "problem",
          "4,49": "problem",
          "4,50": "problem",
          "4,51": "problem",
          "4,52": "problem",
          "4,54": "problem",
          "4,56": "problem",
          "4,68": "problem",
          "4,69": "problem",
          "4,75": "problem",
          "4,80": "problem",
          "4,81": "problem",
          "4,82": "problem",
          "5,0": "treatment",
          "5,2": "problem",
          "5,4": "test",
          "5,5": "test",
          "5,6": "test",
          "5,11": "problem",
          "5,12": "problem",
          "5,16": "problem",
          "5,21": "treatment",
          "5,22": "treatment",
          "5,24": "problem",
          "5,25": "problem",
          "5,26": "problem",
          "5,31": "problem",
          "5,32": "problem",
          "5,36": "treatment",
          "5,37": "treatment",
          "5,38": "treatment",
          "5,43": "treatment",
          "5,44": "treatment",
          "5,57": "test",
          "6,0": "test",
          "6,1": "test",
          "6,7": "problem",
          "6,8": "problem",
          "6,15": "test",
          "6,17": "test",
          "6,18": "test",
          "6,22": "treatment",
          "6,23": "treatment",
          "6,25": "treatment",
          "6,26": "treatment",
          "6,31": "treatment",
          "6,32": "treatment",
          "6,33": "treatment",
          "6,34": "treatment",
          "6,35": "treatment",
          "6,36": "treatment",
          "7,0": "problem",
          "7,1": "problem",
          "7,2": "problem",
          "7,3": "problem",
          "7,5": "problem",
          "7,6": "problem",
          "7,7": "problem",
          "7,8": "problem",
          "7,16": "problem",
          "7,19": "problem",
          "7,22": "problem",
          "7,24": "problem",
          "7,28": "problem",
          "7,29": "problem",
          "7,31": "problem",
          "7,32": "problem",
          "7,33": "problem",
          "7,36": "treatment",
          "7,37": "treatment",
          "7,39": "test",
          "7,40": "test",
          "7,45": "treatment",
          "7,46": "treatment",
          "7,47": "test",
          "7,48": "test",
          "7,51": "test",
          "7,53": "problem",
          "7,54": "problem",
          "7,55": "problem",
          "7,58": "treatment",
          "7,61": "problem",
          "7,63": "test",
          "7,64": "test",
          "7,66": "problem",
          "7,68": "problem",
          "7,69": "problem",
          "7,72": "treatment",
          "7,77": "test",
          "7,78": "test",
          "7,83": "test",
          "7,88": "test",
          "7,92": "test",
          "7,93": "test",
          "7,94": "test",
          "8,1": "test",
          "8,2": "test",
          "8,4": "test",
          "8,13": "test",
          "8,29": "test",
          "8,30": "test",
          "8,36": "treatment",
          "8,52": "test",
          "8,53": "test",
          "8,58": "treatment",
          "8,67": "treatment",
          "8,68": "treatment",
          "8,70": "test",
          "8,71": "test",
          "8,73": "treatment",
          "8,77": "test",
          "8,78": "test",
          "8,91": "test",
          "8,92": "test",
          "8,103": "test",
          "8,109": "test",
          "8,110": "test",
          "8,112": "test",
          "8,115": "problem",
          "8,117": "test",
          "8,118": "test",
          "8,120": "test",
          "9,6": "problem",
          "9,7": "problem",
          "9,17": "problem",
          "9,21": "test",
          "9,22": "test",
          "9,28": "problem",
          "9,30": "problem",
          "9,36": "test",
          "9,37": "test",
          "9,40": "treatment",
          "9,41": "treatment",
          "9,43": "problem",
          "9,44": "problem",
          "9,45": "problem",
          "9,46": "problem",
          "9,49": "problem",
          "9,50": "problem",
          "9,51": "problem",
          "9,57": "problem",
          "9,62": "treatment",
          "9,71": "problem",
          "9,78": "problem",
          "9,80": "problem",
          "9,81": "problem",
          "9,83": "treatment",
          "9,91": "treatment",
          "9,92": "treatment",
          "9,125": "treatment",
          "9,128": "problem",
          "9,129": "problem",
          "9,130": "problem",
          "9,131": "problem",
          "9,132": "problem",
          "9,133": "problem",
          "9,135": "problem",
          "9,136": "problem",
          "9,138": "treatment",
          "9,151": "test",
          "9,163": "treatment",
          "9,164": "treatment",
          "9,166": "problem",
          "9,168": "test",
          "9,169": "test",
          "9,171": "test",
          "9,172": "test",
          "9,173": "test",
          "9,180": "problem",
          "9,181": "problem",
          "9,184": "test",
          "9,185": "test",
          "9,187": "problem",
          "9,188": "problem",
          "9,190": "problem",
          "9,191": "problem",
          "9,192": "problem",
          "9,198": "problem",
          "9,200": "problem",
          "9,201": "problem",
          "9,202": "problem",
          "9,203": "problem",
          "9,207": "treatment",
          "9,218": "treatment",
          "9,220": "problem",
          "9,221": "problem",
          "9,222": "problem",
          "9,223": "problem",
          "9,229": "treatment",
          "9,230": "treatment",
          "9,232": "treatment",
          "9,233": "treatment",
          "9,239": "test",
          "9,240": "test",
          "9,241": "test",
          "9,242": "test",
          "9,250": "treatment",
          "9,251": "treatment",
          "9,252": "treatment",
          "9,264": "test",
          "9,265": "test",
          "9,304": "problem",
          "9,305": "problem",
          "9,311": "treatment",
          "9,312": "treatment",
          "9,313": "treatment",
          "9,314": "treatment",
          "9,315": "test",
          "9,316": "test",
          "9,329": "problem",
          "9,331": "problem",
          "9,332": "problem",
          "9,336": "treatment",
          "9,340": "problem",
          "9,341": "problem",
          "9,344": "test",
          "9,345": "test",
          "9,346": "test",
          "9,350": "treatment",
          "9,352": "test",
          "9,353": "test",
          "9,359": "problem",
          "9,360": "problem",
          "9,362": "problem",
          "9,363": "problem",
          "9,364": "problem",
          "9,366": "test",
          "9,367": "test",
          "9,368": "test",
          "9,370": "test",
          "9,372": "problem",
          "9,374": "treatment",
          "9,375": "treatment",
          "9,376": "treatment",
          "9,377": "test",
          "9,378": "test",
          "9,389": "treatment",
          "9,390": "treatment",
          "9,391": "treatment",
          "9,400": "treatment",
          "9,416": "problem",
          "9,425": "test",
          "9,426": "test",
          "9,434": "treatment",
          "9,436": "treatment",
          "9,448": "treatment",
          "9,449": "treatment",
          "9,450": "test",
          "9,451": "test",
          "9,466": "test",
          "9,467": "test",
          "9,468": "test",
          "9,472": "problem",
          "9,474": "problem",
          "10,13": "problem",
          "10,14": "problem",
          "10,18": "problem",
          "10,19": "problem",
          "10,21": "test",
          "10,22": "test",
          "10,23": "test",
          "10,25": "test",
          "10,27": "test",
          "10,28": "test",
          "10,29": "test",
          "10,30": "test",
          "10,31": "test",
          "10,38": "problem",
          "10,40": "problem",
          "10,46": "problem",
          "10,47": "problem",
          "10,55": "problem",
          "10,56": "problem",
          "10,60": "problem",
          "10,61": "problem",
          "10,68": "problem",
          "10,69": "problem",
          "10,70": "problem",
          "10,71": "problem",
          "10,72": "problem",
          "10,74": "test",
          "10,75": "test",
          "10,85": "test",
          "10,86": "test",
          "10,89": "test",
          "10,90": "test",
          "10,91": "test",
          "10,92": "test",
          "10,97": "test",
          "10,98": "test",
          "10,100": "test",
          "10,101": "test",
          "10,102": "test",
          "10,103": "test",
          "10,105": "test",
          "10,108": "test",
          "10,109": "test",
          "10,113": "test",
          "10,114": "test",
          "10,115": "test",
          "10,116": "test",
          "10,117": "test",
          "10,121": "problem",
          "10,122": "problem",
          "10,126": "test",
          "10,128": "problem",
          "10,129": "problem",
          "10,131": "problem",
          "10,132": "problem",
          "10,133": "problem",
          "10,135": "test",
          "10,136": "test",
          "10,138": "test",
          "10,139": "test",
          "10,143": "test",
          "10,144": "test",
          "10,145": "test",
          "10,149": "test",
          "10,150": "test",
          "10,155": "test",
          "10,156": "test",
          "10,157": "test",
          "10,163": "treatment",
          "10,164": "treatment",
          "10,165": "treatment",
          "11,6": "problem",
          "11,7": "problem",
          "11,23": "problem",
          "11,25": "problem",
          "11,26": "problem",
          "11,32": "test",
          "11,33": "test",
          "11,39": "test",
          "11,40": "test",
          "11,41": "test",
          "11,42": "test",
          "11,45": "problem",
          "11,46": "problem",
          "11,47": "problem",
          "11,49": "problem",
          "11,57": "test",
          "11,58": "test",
          "11,59": "test",
          "11,67": "test",
          "11,68": "test",
          "11,69": "test",
          "11,74": "treatment",
          "11,75": "treatment",
          "11,76": "treatment",
          "11,77": "treatment",
          "11,78": "treatment",
          "11,79": "treatment",
          "11,80": "treatment",
          "11,81": "treatment",
          "11,82": "treatment",
          "11,84": "treatment",
          "11,85": "treatment",
          "11,86": "treatment",
          "11,88": "treatment",
          "11,89": "treatment",
          "11,92": "treatment",
          "11,93": "treatment",
          "11,94": "treatment",
          "11,95": "treatment",
          "11,96": "treatment",
          "11,97": "treatment",
          "11,101": "test",
          "11,102": "test",
          "11,105": "problem",
          "11,106": "problem",
          "11,107": "problem",
          "11,108": "problem",
          "11,109": "problem",
          "11,111": "problem",
          "11,112": "problem",
          "11,120": "treatment",
          "11,121": "treatment",
          "11,122": "treatment",
          "11,125": "problem",
          "11,127": "problem",
          "11,128": "problem",
          "11,130": "test",
          "11,131": "test",
          "11,136": "test",
          "11,137": "test",
          "11,139": "test",
          "11,144": "test",
          "11,145": "test",
          "11,146": "test",
          "11,147": "test",
          "11,148": "test",
          "11,150": "test",
          "11,151": "test",
          "11,152": "test",
          "11,153": "test",
          "11,154": "test",
          "11,157": "test",
          "11,158": "test",
          "11,159": "test",
          "11,161": "test",
          "11,162": "test",
          "11,163": "test",
          "11,164": "test",
          "11,165": "test",
          "11,167": "problem",
          "11,168": "problem",
          "11,175": "test",
          "11,176": "test",
          "11,177": "test",
          "12,6": "problem",
          "12,7": "problem",
          "12,9": "problem",
          "12,10": "problem",
          "12,12": "test",
          "12,14": "test",
          "12,15": "test",
          "12,16": "test",
          "12,18": "problem",
          "12,19": "problem",
          "12,26": "treatment",
          "12,27": "treatment",
          "13,0": "test",
          "13,1": "test",
          "13,3": "problem",
          "13,19": "treatment",
          "13,20": "treatment",
          "13,26": "treatment",
          "13,27": "treatment",
          "13,37": "problem",
          "13,38": "problem",
          "13,42": "problem",
          "13,45": "test",
          "13,46": "test",
          "13,47": "test",
          "14,0": "test",
          "14,1": "test",
          "14,2": "test",
          "14,3": "test",
          "14,4": "test",
          "14,5": "test",
          "14,6": "test",
          "14,7": "test",
          "14,14": "test",
          "14,15": "test",
          "14,23": "test",
          "14,24": "test",
          "14,25": "test",
          "14,29": "test",
          "14,30": "test",
          "14,31": "test",
          "14,32": "test",
          "14,33": "test",
          "14,43": "problem",
          "14,44": "problem",
          "14,49": "test",
          "14,50": "test",
          "14,53": "test",
          "14,54": "test",
          "14,55": "test",
          "14,56": "test",
          "14,65": "test",
          "14,68": "test",
          "14,69": "test",
          "14,70": "test",
          "14,78": "test",
          "14,80": "problem",
          "14,84": "test",
          "14,85": "test",
          "14,86": "test",
          "14,87": "test",
          "14,88": "test",
          "14,90": "test",
          "14,91": "test",
          "14,92": "test",
          "14,97": "problem",
          "14,99": "test",
          "14,100": "test",
          "14,102": "problem",
          "14,103": "problem",
          "14,104": "problem",
          "14,106": "test",
          "14,107": "test",
          "14,114": "test",
          "14,115": "test",
          "14,118": "test",
          "14,119": "test",
          "14,131": "test",
          "14,132": "test",
          "14,135": "problem",
          "14,136": "problem",
          "14,138": "problem",
          "14,139": "problem",
          "14,143": "test",
          "14,144": "test",
          "14,149": "test",
          "14,152": "problem",
          "14,155": "problem",
          "14,158": "treatment",
          "14,159": "treatment",
          "14,160": "treatment",
          "14,162": "treatment",
          "14,163": "treatment",
          "14,164": "treatment",
          "14,165": "treatment",
          "14,168": "problem",
          "14,169": "problem",
          "14,170": "problem",
          "14,175": "problem",
          "14,176": "problem",
          "14,177": "problem",
          "14,178": "problem",
          "14,183": "problem",
          "14,186": "problem",
          "14,187": "problem",
          "14,192": "test",
          "14,194": "problem",
          "14,195": "problem",
          "14,196": "problem",
          "14,199": "problem",
          "14,200": "problem",
          "15,0": "test",
          "15,1": "test",
          "15,2": "test",
          "15,3": "test",
          "15,10": "test",
          "15,11": "test",
          "15,12": "treatment",
          "15,17": "test",
          "15,18": "test",
          "15,20": "test",
          "15,21": "test",
          "15,22": "test",
          "15,23": "test",
          "15,24": "test",
          "15,25": "test",
          "15,28": "test",
          "15,29": "test",
          "15,30": "test",
          "15,33": "treatment",
          "15,36": "treatment",
          "15,37": "treatment",
          "15,38": "treatment",
          "15,39": "treatment",
          "15,40": "treatment",
          "15,41": "treatment",
          "15,44": "treatment",
          "15,45": "treatment",
          "15,46": "treatment",
          "15,47": "treatment",
          "15,48": "treatment",
          "15,49": "treatment",
          "15,51": "test",
          "15,52": "test",
          "15,53": "test",
          "15,57": "treatment",
          "15,58": "treatment",
          "15,62": "test",
          "15,63": "test",
          "15,65": "treatment",
          "15,66": "treatment",
          "15,67": "treatment",
          "15,68": "treatment",
          "15,70": "treatment",
          "15,71": "treatment",
          "15,72": "treatment",
          "15,73": "treatment",
          "16,0": "test",
          "16,1": "test",
          "16,13": "problem",
          "16,14": "problem",
          "16,15": "problem",
          "16,18": "problem",
          "16,19": "problem",
          "16,21": "test",
          "16,22": "test",
          "16,23": "test",
          "16,24": "test",
          "16,25": "test",
          "16,26": "test",
          "16,28": "test",
          "16,29": "test",
          "16,30": "test",
          "16,33": "test",
          "16,34": "test",
          "16,35": "test",
          "16,36": "test",
          "16,37": "test",
          "16,40": "problem",
          "16,41": "problem",
          "16,42": "problem",
          "16,49": "test",
          "16,50": "test",
          "16,51": "test",
          "16,52": "test",
          "16,54": "problem",
          "16,56": "test",
          "16,58": "test",
          "16,59": "test",
          "16,68": "test",
          "16,69": "test",
          "16,70": "test",
          "16,72": "test",
          "16,73": "test",
          "16,86": "problem",
          "16,87": "problem",
          "16,88": "problem",
          "16,90": "problem",
          "16,91": "treatment",
          "16,92": "treatment",
          "16,93": "treatment",
          "16,94": "treatment",
          "16,101": "test",
          "16,103": "problem",
          "16,106": "problem",
          "16,108": "problem",
          "16,115": "test",
          "16,116": "test",
          "16,117": "test",
          "16,118": "test",
          "16,120": "test",
          "16,121": "test",
          "16,122": "test",
          "16,124": "test",
          "16,125": "test",
          "16,126": "test",
          "16,127": "test",
          "16,129": "test",
          "16,130": "test",
          "16,134": "test",
          "16,135": "test",
          "16,139": "problem",
          "16,140": "problem",
          "16,141": "problem",
          "16,142": "problem",
          "16,143": "problem",
          "16,144": "problem",
          "16,147": "test",
          "16,148": "test",
          "16,149": "test",
          "16,150": "test",
          "16,152": "test",
          "16,158": "test",
          "16,159": "test",
          "16,160": "test",
          "16,165": "problem",
          "16,174": "treatment",
          "16,175": "treatment",
          "16,176": "treatment",
          "16,179": "test",
          "16,180": "test",
          "16,181": "test",
          "16,184": "test",
          "16,185": "test",
          "16,188": "treatment",
          "16,189": "treatment",
          "16,194": "treatment",
          "16,195": "treatment",
          "16,196": "treatment",
          "16,197": "treatment",
          "16,198": "treatment",
          "16,199": "treatment",
          "16,210": "test",
          "16,211": "test",
          "17,0": "test",
          "17,1": "test",
          "17,2": "test",
          "17,3": "test",
          "17,9": "test",
          "17,10": "test",
          "17,14": "test",
          "17,15": "test",
          "17,16": "test",
          "17,17": "test",
          "17,18": "test",
          "17,19": "test",
          "17,20": "test",
          "17,21": "test",
          "17,22": "test",
          "17,25": "test",
          "17,26": "test",
          "17,28": "test",
          "17,29": "test",
          "17,32": "problem",
          "17,33": "problem",
          "17,35": "test",
          "17,44": "test",
          "17,45": "test",
          "17,46": "test",
          "17,48": "problem",
          "17,49": "problem",
          "17,52": "problem",
          "17,54": "problem",
          "17,55": "problem",
          "17,61": "problem",
          "17,62": "problem",
          "17,67": "test",
          "17,69": "test",
          "17,70": "test",
          "17,72": "test",
          "17,73": "test",
          "17,75": "test",
          "17,76": "test",
          "17,77": "test",
          "17,81": "test",
          "17,82": "test",
          "17,86": "test",
          "17,87": "test",
          "17,90": "test",
          "17,91": "test",
          "17,92": "test",
          "17,94": "test",
          "17,95": "test",
          "17,97": "treatment",
          "18,1": "test",
          "18,2": "test",
          "18,5": "problem",
          "18,8": "problem",
          "18,23": "problem",
          "18,26": "test",
          "18,27": "test",
          "18,28": "test",
          "18,29": "test",
          "18,30": "test",
          "18,33": "test",
          "18,34": "test",
          "18,35": "test",
          "18,37": "test",
          "18,38": "test",
          "18,41": "test",
          "18,42": "test",
          "18,43": "test",
          "18,44": "test",
          "18,45": "test",
          "18,46": "test",
          "18,47": "test",
          "18,52": "problem",
          "18,53": "problem",
          "18,55": "test",
          "18,56": "test",
          "18,58": "test",
          "18,63": "problem",
          "18,64": "problem",
          "18,65": "problem",
          "18,69": "test",
          "18,70": "test",
          "18,74": "test",
          "18,75": "test",
          "18,76": "test",
          "18,77": "test",
          "18,78": "test",
          "18,85": "problem",
          "18,87": "problem",
          "18,89": "problem",
          "18,91": "test",
          "18,92": "test",
          "18,95": "problem",
          "18,96": "problem",
          "18,105": "problem",
          "18,112": "problem",
          "18,113": "problem",
          "18,114": "problem",
          "18,115": "problem",
          "19,1": "test",
          "19,4": "problem",
          "19,11": "test",
          "19,12": "test",
          "19,13": "test",
          "19,16": "problem",
          "19,18": "test",
          "19,19": "test",
          "19,20": "test",
          "19,21": "test",
          "19,22": "test",
          "19,26": "test",
          "19,27": "test",
          "19,29": "test",
          "19,30": "test",
          "19,32": "test",
          "19,33": "test",
          "19,35": "test",
          "19,36": "test",
          "19,38": "test",
          "19,39": "test",
          "19,40": "test",
          "19,41": "test",
          "19,43": "test",
          "19,45": "test",
          "19,46": "test",
          "19,47": "test",
          "19,48": "test",
          "19,52": "problem",
          "19,53": "problem",
          "19,54": "problem",
          "19,56": "test",
          "19,57": "test",
          "19,58": "test",
          "19,59": "test",
          "19,60": "test",
          "19,62": "test",
          "19,63": "test",
          "19,65": "test",
          "19,66": "test",
          "19,70": "test",
          "19,71": "test",
          "19,72": "test",
          "19,73": "test",
          "19,74": "test",
          "19,75": "test",
          "19,76": "test",
          "19,77": "test",
          "19,78": "test",
          "19,79": "test",
          "19,85": "problem",
          "19,86": "problem",
          "19,88": "test",
          "19,89": "test",
          "19,90": "test",
          "19,92": "problem",
          "19,93": "problem",
          "19,95": "problem",
          "19,96": "problem",
          "19,97": "problem",
          "19,98": "problem",
          "19,104": "problem",
          "19,105": "problem",
          "19,106": "problem",
          "19,107": "problem",
          "19,111": "problem",
          "19,113": "test",
          "19,114": "test",
          "19,115": "test",
          "19,118": "test",
          "19,119": "test",
          "19,120": "test",
          "19,122": "problem",
          "19,133": "problem",
          "19,134": "problem",
          "19,135": "problem",
          "19,136": "problem",
          "19,137": "problem",
          "19,138": "problem",
          "19,142": "test",
          "19,143": "test",
          "19,147": "test",
          "19,151": "test",
          "20,0": "test",
          "20,1": "test",
          "20,2": "test",
          "20,7": "problem",
          "20,8": "problem",
          "20,19": "test",
          "20,20": "test",
          "20,22": "test",
          "20,23": "test",
          "20,24": "test",
          "20,29": "problem",
          "20,32": "problem",
          "20,33": "problem",
          "20,34": "problem",
          "20,35": "problem",
          "20,40": "problem",
          "20,42": "problem",
          "20,43": "problem",
          "20,47": "treatment",
          "20,48": "treatment",
          "20,49": "treatment",
          "21,4": "problem",
          "21,26": "treatment",
          "21,28": "problem",
          "21,29": "problem",
          "21,30": "problem",
          "21,31": "problem",
          "21,33": "problem",
          "21,34": "problem",
          "21,35": "problem",
          "21,38": "test",
          "21,47": "treatment",
          "21,51": "problem",
          "21,52": "problem",
          "21,54": "problem",
          "21,55": "problem",
          "22,6": "problem",
          "22,7": "problem",
          "22,10": "treatment",
          "22,11": "treatment",
          "22,20": "treatment",
          "22,28": "treatment",
          "22,29": "treatment",
          "22,30": "treatment",
          "22,38": "treatment",
          "22,39": "treatment",
          "22,40": "treatment",
          "22,51": "problem",
          "22,61": "problem",
          "22,62": "problem",
          "22,63": "problem",
          "22,64": "problem",
          "22,66": "problem",
          "22,67": "problem",
          "22,68": "problem",
          "22,69": "problem",
          "22,70": "problem",
          "22,72": "problem",
          "22,73": "problem",
          "22,77": "treatment",
          "22,78": "treatment",
          "22,82": "treatment",
          "22,83": "treatment",
          "22,86": "problem",
          "22,87": "problem",
          "22,93": "treatment",
          "22,94": "treatment",
          "22,109": "treatment",
          "22,110": "treatment",
          "22,122": "test",
          "22,123": "test",
          "22,134": "treatment",
          "22,135": "treatment",
          "22,136": "treatment",
          "22,144": "treatment",
          "22,145": "treatment",
          "22,155": "problem",
          "22,156": "problem",
          "22,157": "problem",
          "22,159": "problem",
          "22,160": "problem",
          "22,161": "test",
          "22,162": "test",
          "22,164": "treatment",
          "22,165": "treatment",
          "22,166": "treatment",
          "22,168": "treatment",
          "22,169": "treatment",
          "22,170": "treatment",
          "22,174": "problem",
          "22,176": "problem",
          "22,178": "problem",
          "22,185": "treatment",
          "22,186": "treatment",
          "22,191": "problem",
          "22,192": "problem",
          "22,193": "problem",
          "22,194": "problem",
          "22,196": "treatment",
          "22,197": "treatment",
          "22,198": "treatment",
          "22,199": "treatment",
          "22,204": "problem",
          "22,205": "problem",
          "22,207": "problem",
          "22,208": "problem",
          "22,209": "problem",
          "22,214": "problem",
          "22,215": "problem",
          "22,232": "problem",
          "22,234": "problem",
          "22,235": "problem",
          "22,236": "problem",
          "22,243": "treatment",
          "22,244": "treatment",
          "22,245": "treatment",
          "22,246": "treatment"
        }
      }
    },
    url:
      "https://www.ncbi.nlm.nih.gov/pubmed/16762539/; https://api.elsevier.com/content/article/pii/S0956566306002296; https://www.sciencedirect.com/science/article/pii/S0956566306002296"
  },
  {
    answer: {
      score: "96.63428994184085",
      sents: [
        "In addition to direct government contributions, innovative finance mechanisms have been successful in raising funds for vaccines in the past and should be used to fund the development of COVID-19 vaccines.",
        "8, 9 The International Finance Facility for Immunisation (IFFIm) raises funds with vaccine bonds, which turn long-term contributions by donors into available cash.",
        "8 IFFIm was created to support Gavi, the Vaccine Alliance, but could be used to finance CEPI's COVID-19 vaccine efforts.",
        "With advanced market commitments, donors make funding commitments to vaccine manufacturers and, in exchange, companies sign a legally binding commitment to provide the vaccines at a price affordable to low-income and middle-income countries.",
        "Gavi's board expressed support for the use of Gavi's IFFIm and advanced market commitments to improve COVID-19 vaccine development and access.",
        "10 The need for COVID-19 vaccines is global, although the need is differentially distributed within populations.",
        "Vaccines would likely be prioritised for health-care workers and people at greatest risk of severe illness and death.",
        "High-income countries must not monopolise the global supply of COVID-19 vaccines.",
        "This risk is real: during the 2009 influenza A/H1N1 pandemic, rich countries negotiated large advance orders for the vaccine, crowding out poor countries.",
        "11 Such an outcome would result in a suboptimal allocation of an initially scarce resource."
      ]
    },

    paper_id: "d0dafaa2700d9fa0289b620ce87992f9cf393ae1",
    doi: "10.1016/s0140-6736(20)30763-7",
    doc_date: "2020-03-31",
    url: "https://doi.org/10.1016/j.bbrc.2020.03.047",
    title:
      "Spike protein recognition of mammalian ACE2 predicts the host range and an optimized ACE2 for SARS-CoV-2 infection",
    authors: ["Junwen.Luan", "Yue.Lu", "Xiaolu.Jin", "Leiliang.Zhang"],
    summary: "",
    abstract: {
      text: [
        "a b s t r a c t SARS-CoV-2 causes the recent global COVID-19 public health emergency",
        " ACE2 is the receptor for both SARS-CoV-2 and SARS-CoV",
        " To predict the potential host range of SARS-CoV-2, we analyzed the key residues of ACE2 for recognizing S protein",
        " We found that most of the selected mammals including pets (dog and cat), pangolin and Circetidae mammals remained the most of key residues for association with S protein from SARS-CoV and SARS-CoV-2",
        " The interaction interface between cat/dog/pangolin/Chinese hamster ACE2 and SARS-CoV/SARS-CoV-2 S protein was simulated through homology modeling",
        " We identified that N82 in ACE2 showed a closer contact with SARS-CoV-2 S protein than M82 in human ACE2",
        " Our finding will provide important insights into the host range of SARS-CoV-2 and a new strategy to design an optimized ACE2 for SARS-CoV-2 infection",
        "\nPlease cite this article as: J",
        " Luan et al",
        ", Spike protein recognition of mammalian ACE2 predicts the host range and an optimized ACE2 for SARS-CoV-2 infection, Biochemical and Biophysical Research Communications, https://doi"
      ],
      tags: {
        sciwing: [
          "background",
          "finding",
          "method",
          "finding",
          "finding",
          "finding",
          "finding",
          "other",
          "other",
          "finding"
        ]
      }
    },
    bodyText: {
      text: [
        "Corona Virus Disease 2019 (COVID- 19) , which was reported from Wuhan city, Hubei province of China, has caused over 78,000 human infections and more than 2700 deaths (as of February 25, 2020) [1, 2] . Severe Acute Respiratory Syndrome Corona Virus 2 (SARS-CoV-2) was identified as the pathogen of COVID- 19 [1e3] . After SARS-CoV and MERS-CoV, SARS-CoV-2 has become the third coronavirus that causes severe respiratory disease and human death [4, 5] .",
        "Belonging to the subgenus sarbecvirus of Coronaviridae, both SARS-CoV-2 and SARS-CoV are human SARS-related coronavirus (SARSr-CoV). Its genome is a single-stranded RNA composed of about 30 kb nucleotides. SARS-CoV-2 encodes at least four major structural proteins, namely spike protein (S), membrane protein (M), envelope protein (E), and nucleocapsid protein (N) [6] . S protein, which is a type I glycoprotein, protrudes from the surface of the virus and can contact the host cell earlier. S protein has attracted great attention because of its function in receptor binding.",
        "Angiotensin-converting enzyme 2 (ACE2) binds to the receptorbinding motif (RBM) in the receptor-binding domain (RBD) of SARS-CoV and functions as a receptor for SARS-CoV [7, 8] . ACE2 is widely distributed in heart, liver, testis, kidney, intestine and other tissues. It has the physiological functions of regulating heart and kidney function and controlling blood pressure [9] . Recently, it has been found that human ACE2 promoted the entry of SARS-CoV-2 into the cells [3, 10] . RBD domain of SARS-CoV-2 interacts with human ACE2. Thus, ACE2 is defined as the receptor for SARS-CoV-2.",
        "The specificity of the interaction between virus and receptor determines the host tropism and host range. The origin of SARS-CoV-2 is presumed to be bat [3] . However, the intermediate host is not clear, and some studies suggest that pangolin is involved in the evolution of SARS-CoV-2 [11, 12] . It is not clear which mammals are involved in the evolution of SARS-CoV-2 and which animals may be infected by SARS-CoV-2. By sequence alignment of key amino acids binding to RBD in ACE2, the interaction between RBD of SARS-CoV-2/SARS-CoV and mammalian ACE2 was predicted. Based on the potential interaction between S protein and mammalian ACE2, it was speculated that SARS-CoV-2/SARS-CoV preserved the ability to infect many mammals including cat, dog, pangolin and Chinese hamster. From the structure stimulation, we identified N82 in ACE2 show closer contact with F486 of SARS-CoV-2 S protein compared with M82 of ACE2.",
        "The S protein sequence of SARS-CoV-2 is YP_009724390.1, and the S protein sequence of SARS-CoV is NP_828851.1. RBM of SARS-CoV is from 424 to 494. RBM of SARS-CoV-2 is from 437 to 508.",
        "A total of 42 mammalian ACE2 protein sequences were selected from the wild animal protection lists of Hubei Province and Jiangxi Province, primates, bats, dog, and cat. These sequences are as follows: hACE2: Homo sapiens (BAB40370. Cavia porcellus (XP_023417808.1), CgACE2: Cricetulus griseus (A0A061HZ66). Based on the known key sites in SARS-CoV S protein interacting with human ACE2, we analyzed whether these sites were conserved on ACE2 from wild mammals and domestic pets. Phylogenetic and molecular evolutionary analyses of ACE2 were conducted using MEGA version X [13] . Phylogenetic tree was generated with Jones-Taylor-Thornton (JTT) evolutionary model using Maximum Likelihood method.",
        "The interaction interfaces of SARSr-CoV S and ACE2 from cat/ dog/pangolin/Chinese hamster were simulated by Chimera software Ver 1.14 [14] . The simulation were based on the structures of hACE2 with SARS-CoV S RBD (PDB: 2AJF) [7] and hACE2 with SARS-CoV-2 S RBD (PDB: 6LZG).",
        "Human ACE2 is the receptor for both SARS-CoV and SARS-CoV-2. According to the literature [15] , the key amino acids (AAs) in the S protein of SARS-CoV interacting with human ACE2 are Y442, L472, N479, D480, T487, Y491 [7, 15] . We compared the RBM in S protein of SARS-CoV-2 with that of SARS-CoV (Fig. 1A) . These amino acids corresponding to SARS-CoV-2 are L455, F486, Q493, S494, N501 and Y505 (Fig. 1A) . Although five of six key AAs in SARS-CoV-2 are changed compared with SARS-CoV, the overall structure of interfaces of ACE2-RBM are similar (Fig. 1B) .",
        "The key AAs in hACE2 for interacting with RBM are K31, E35, D38, M82 and K353 [7, 15] . Among them, K31 and K353 in hACE2 are most critical residues for RBM recognition. Because the overall structure of interfaces of ACE2-RBM in SARS-CoV-2 and SARS-CoV are similar, we analyzed the key RBD recognizing AAs of ACE2 protein from selected mammals, as shown in Table 1 . We predicted that the mammals whose ACE2 could bind to the S protein of SARS- Next, we constructed a phylogenetic tree for mammalian ACE2 proteins. The ACE2 protein sequences of 42 mammalian animals were compared by ClustalW method of MEGA-X software. Then the JTT model of maximum likelihood method was used to construct ACE2 phylogenetic tree. As shown in Fig. 2 , species that cannot bind to S protein are marked in red, and species that can bind to S protein are marked in green. No correlation between genetic distance and the interaction of ACE2/S was found. Some Pets including dog (Canis lupus familiaris) and cat (Felis catus) potentially recognize S protein, indicating the importance to monitor the pets for SARS-CoV-2 infection. We found that four members of Circetidae including Mesocricetus auratus, Phodopus campbelli, Ictidomys tridecemlineatus, and Cricetulus griseus remained the key residues for association with S protein from SARS-CoV and SARS-CoV-2, though two members of Muridae (Rattus norvegicus, Mus musculus) could not bind to S protein. This founding suggested that Circetidae mammals could be developed as SARSr-CoV small animal models.",
        "We noticed that ACE2 from dog, cat, pangolin and Chinese hamster potentially associated with S protein (Table 1 ). Next, we predicted the structure of cat/dog/pangolin/Chinese hamster ACE2 with SARSr-CoV RBD. The structure of protein complex between RBD region of S protein of SARS-CoV and human ACE2 has been resolved (PDB: 2AJF) [7] . Recently, the structure of SARS-CoV-2 RBD with human ACE2 was also determined [16, 17] . We used Chimera software to display homologous model, and obtained the interaction complex structure of RBD region of SARSr-CoV (SARS-CoV-2 and SARS-CoV) and cat/dog/pangolin/Chinese hamster ACE2. Overall, the RBM structures of S protein of SARS-CoV-2 and SARS-CoV are similar (Fig. 3) . Interaction interface of SARSr-CoV RBD and cat/dog/pangolin/Chinese hamster ACE2 confirmed the potential interaction between SARS-CoV-2 and cat/dog/pangolin/ Chinese hamster ACE2, indicating that these ACE2 could support SARS-CoV-2 entry. The AA in 82 position of human ACE2 is M82, while the corresponding AA in cat, dog, pangolin and Chinese hamster ACE2 is T82, T81, N82 and N82, respectively. The distance between F486 of SARS-CoV-2 S protein and the corresponding AA in ACE2 is 3.753Å for human (Fig. 1A) , 2.695Å for dog (Fig. 3A) , 3.753Å for cat (Fig. 3B ), 1.621Å for pangolin (Fig. 3C) , and 2.024Å for Chinese hamster (Fig. 3D) , respectively. We concluded that N82 in ACE2 showed closer contact with F486 of SARS-CoV-2 S protein than M82 of ACE2.",
        "The host tropism of zoonotic coronavirus is hybrid, and it is important to determine the natural host and host range of coronavirus. In the past two decades, SARS-CoV, MERS-CoV and SARS-CoV-2 have caused serious outbreaks of human infectious diseases. All the three human coronaviruses originated from bats, but the intermediate hosts were different. SARS-CoV is believed to come from the Paguma larvata [18] , and the intermediate host of MERS-CoV is Camelus dromedaries [19] . The new coronavirus SARS-CoV-2 has recently caused a serious epidemic in China, but its intermediate hosts are not clear.",
        "S protein of SARS-CoV-2 interacts with human ACE2, which promotes the entry of SARS-CoV-2, indicating that human ACE2 is the receptor of SARS-CoV-2 [3] . ACE2 contains at least five key amino acids critical for binding S protein of SARSr-CoV [15] . Based on these five amino acids, we analyzed the corresponding amino Table 1 Prediction of the RBD binding capacity of mammalian ACE2.",
        "AA position  matched AA  binding capacity   31  35  38  82  353 hACE2",
        "Phylogenetic tree of mammalian ACE2 proteins. ACE2 sequences from a total of 42 mammals were analyzed by MEGA-X and the phylogenetic tree was constructed with JTT evolutionary model using Maximum Likelihood method. The red represents the species whose ACE2 cannot bind to S protein, and the green is the species whose ACE2 associate with S protein. (For interpretation of the references to colour in this figure legend, the reader is referred to the Web version of this article.)",
        "acids of different mammals to determine which mammalian ACE2 could interact with S protein of human SARSr-CoV. By analyzing the protein sequence of mammalian ACE2, we found that the ACE2 of Camelus dromedarius, Procyon lotor, Rhinolophus ferrumequinum, Rattus norvegicus, Mus musculus, Ornithorhynchus anatinus, Loxodonta africana, Erinaceus europaeus, Nyctereutes procyonoides, Suricata suricatta, Dipodomys ordii, and Cavia porcellus lose the capability to associate with S protein (Table 1) . These mammals could be ruled out from the potential host list for SARS-CoV-2. We found that S protein may bind to ACE2 from some wild mammals, which suggests that we should investigate whether these animals may be intermediate hosts for SARS-CoV-2. It has been reported that the RBM region in S protein of pangolin coronavirus is similar to that of S protein of SARS-CoV-2 [11, 12] , which may be involved in the recombination of SARS-CoV-2. We identified that N82 of pangolin ACE2 showed closer contact with RBD than human ACE2 (Fig. 3C) , indicating that pangolin ACE2 might show better affinity to SARS-CoV-2. This finding further supports the hypothesis that pangolin is involved in SARS-CoV-2 evolution. In current study, only a limited list of wild mammals is covered. In the future, we should select more mammals for study. Although no SARS-CoV-2 has been found in domestic cats and dogs, cat/dog ACE2 may bind to S protein of SARS-CoV-2. In the future, we should pay attention to monitoring whether domestic cats and dogs could be infected by SARS-CoV-2. Animal model is an important tool in the study of infectious diseases. ACE2 of mice cannot interact with SARS-CoV-2, so it cannot be used as animal model of SARS-CoV-2 directly. Some studies have generated mice transfected with human ACE2 as the models to study SARS-CoV [20] , and these mice can also be used as animal models for SARS-CoV-2 infection. Interestingly, we identified that N82 in ACE2 is closer to RBD than M82 (Fig. 3C and D) , indicating a novel strategy to design an optimized ACE2 for SARS-CoV-2 infection. We speculated that small peptide based on N82 of ACE2 might show higher affinity to SARS-CoV-2 RBD. We proposed that if M82 in human ACE2 was mutated to N82, the modified human ACE2 will enhance SARS-CoV-2 infection. In the future, those ideas will be tested in cell culture and animal model. We noticed that the ACE2 proteins from Circetidae (Mesocricetus auratus, Phodopus campbelli, Ictidomys tridecemlineatus, and Cricetulus griseus) are capable to recognize RBD. Mesocricetus auratus (golden hamster) and Cricetulus griseus (Chinese hamster) are experimental animals and our finding indicates the possibility to develop small animal models for SARS-CoV-2 infection using Chinese hamster and golden hamster.",
        "The authors declare that there are no conflicts of interest."
      ],
      section_header: {
        original: [
          "intro",
          "intro",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "conclusion",
          "conclusion",
          "conclusion",
          "conclusion"
        ],
        generic: [
          "intro",
          "intro",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "conclusion",
          "conclusion",
          "conclusion",
          "conclusion"
        ]
      },
      tags: {
        sciwingI2B2: {
          "1,1": "treatment",
          "1,3": "test",
          "3,3": "problem",
          "5,1": "treatment",
          "5,3": "test",
          "4,3": "problem"
        }
      }
    },
    similar: [
      {
        paper_id: "paper id test 1",
        doi: "doi test",
        doc_date: "",
        title:
          "Biophysical characterization of the SARS-CoV-2 spike protein binding with the ACE2 receptor and implications for infectivity",
        authors: ["author test"],
        summary:
          "We computationally contrasted the binding interactions between human ACE2 and coronavirus spike protein receptor binding domain (RBD) of the 2002 epidemic-causing SARS-CoV-1, SARS-CoV-2, and bat coronavirus RaTG13 using the Rosetta energy function.\nWe find that the RBD of the spike protein of SARS-CoV-2 is highly optimized to achieve very strong binding with human ACE2 (hACE2) which is consistent with its enhanced infectivity.\nNotably, the SARS-CoV-2 RBD out-competes the angiotensin 2 receptor type I (ATR1) which is the native binding partner of ACE2 by 35% in terms of the calculated binding affinity.\nBy contrasting the spike protein SARS-CoV-2 Rosetta binding energy with ACE2 of different livestock and pet species we find strongest binding with bat ACE2 followed by human, feline, equine, canine and finally chicken.",
        abstract: {
          text: ["abstract text test"],
          tags: {
            sciwing: ["tag 1"]
          }
        },
        bodyText: {
          text: ["bodytext text test", "paragraph 2"],
          tags: {
            "0": {
              tagPerPara: {
                "0": "Label1",
                "1": "Label2"
              }
            },
            "1": {
              tagPerPara: {
                "0": "Label3",
                "1": "Label4"
              }
            }
          }
        }
      }
    ]
  },
  {
    answer: {
      score: "75.31819711516434",
      sents: [
        "From the insight of glucose metabolism disorder: Oxygen therapy and blood glucose monitoring are crucial for quarantined COVID-19 patients",
        "Majority of the COVID-19 patients in China have been cured following the ever-improving treatment methods, and oxygen supplement has been applied as a common treatment strategy for cases in hospitals and cabin hospitals (China National Health Commission).",
        "Since 81% of the confirmed cases are in mild condition with non-or mild pneumonia, most of COVID-19 patients outside China are now recommended for self-cured under home quarantine.",
        "However, it may only take 9 days for the mild COVID-19 patients without effective nursing to develop into serious or critical cases with acute respiratory distress syndrome (ARDS) (Guan et al., 2020; Huang et al., 2020) .",
        "Therefore, it is critical to promote the self-cure rate of the mild cases and prevent the disease from deterioration.",
        "Considering the harmful consequences induced by hypoxemia and advantages of oxygen therapy in controlling glucose metabolism (Handy et al., 2005) , we suggest that adequate oxygen intake is also essential for the newly confirmed or suspected cases under home quarantine."
      ]
    },
    paper_id: "PMC7126624",
    doi: "10.1016/j.bios.2006.04.024",
    title:
      "DNA hybridization sensor based on aurothiomalate electroactive label on glassy carbon electrodes",
    doc_date: "2007-01-15",
    authors: [
      "Alfredo.de la Escosura-Muñiz",
      "María.González-García",
      "Agustín.Costa-García"
    ],
    summary: "",
    abstract: {
      text: [""],
      tags: {
        sciwing: []
      }
    },
    bodyText: {
      section_header: {
        original: [
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Introduction",
          "Apparatus and electrodes ::: Experimental",
          "Apparatus and electrodes ::: Experimental",
          "Apparatus and electrodes ::: Experimental",
          "Apparatus and electrodes ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Reagents and solutions ::: Experimental",
          "Labeling of the target strand with sodium aurothiomalate ::: Methods ::: Experimental",
          "Labeling of the target strand with sodium aurothiomalate ::: Methods ::: Experimental",
          "Electrode pre-treatment ::: Methods ::: Experimental",
          "Immobilization of the probe ::: Methods ::: Experimental",
          "Hybridization reaction ::: Methods ::: Experimental",
          "Hybridization reaction ::: Methods ::: Experimental",
          "Recording the analytical signal: catalytic effect of gold on the silver electrodeposition ::: Methods ::: Experimental",
          "Recording the analytical signal: catalytic effect of gold on the silver electrodeposition ::: Methods ::: Experimental",
          "Recording the analytical signal: catalytic effect of gold on the silver electrodeposition ::: Methods ::: Experimental",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Single strand DNA labeling ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Immobilization of the probe ::: Results and discussion",
          "Hybridization reaction ::: Results and discussion",
          "Hybridization reaction ::: Results and discussion",
          "Hybridization reaction ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Analytical characteristics of the DNA hybridization sensor ::: Results and discussion",
          "Conclusions",
          "Conclusions",
          "Conclusions",
          "Conclusions",
          "Conclusions",
          "Conclusions",
          "Conclusions"
        ],
        generic: [
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "introduction",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "method",
          "conclusions",
          "conclusions",
          "conclusions",
          "conclusions",
          "conclusions",
          "conclusions",
          "conclusions"
        ]
      },
      text: [
        "DNA diagnostic has become an important area of molecular biology and biotechnology studies. The detection of specific base sequences in human, viral and bacterial nucleic acid is becoming increasingly important in several areas, with applications ranging from the detection of disease-causing and food-contaminating organisms to forensic and environmental research.",
        "Conventional methods for the analysis of specific gene sequences are based on either direct sequencing or DNA hybridization methods. Although the DNA sequencing has undergone very big progress in the last decade (Kling, 2003, Kartalov and Quake, 2004, Aborn et al., 2005), these traditional methods, based on the coupling of electrophoretic separations and radioisotopic (32P) detection, are generally labour intensive and time consuming. Therefore, the second option is more commonly used in diagnostic laboratories.",
        "Genosensors (or DNA hybridization biosensors) offer a promising alternative to carry out the specific gene sequence identification (De los Santos-Álvarez et al., 2004, Wang, 2002, Paleček et al., 1998). These biosensors commonly rely on the immobilization of a single-stranded (ss) oligonucleotide probe onto a transducer surface to recognize – by hybridization – its complementary target sequence.",
        "Among the different kind of transducers found in the literature – chemiluminescent (Nguyen and Heffelfinger, 1995), quartz crystal microbalance (Hashimoto et al., 1994, Steel et al., 1998), fiber optical (Piunno et al., 1995), evanescent wave (Watts et al., 1995) or an-acoustic wave (Su et al., 1996) – those based on electrochemical transduction, due to their high sensitivity, small dimensions, low cost and compatibility with micro-fabrication technology provide a very interesting alternative.",
        "These electrochemical biosensing devices can monitor sequence-specific hybridization processes: (i) directly, based on the intrinsic DNA electroactivity (Kerman et al., 2005, Erdem et al., 2004, Ozkan et al., 2002, Lucarelli et al., 2002) or (ii) indirectly by measuring changes in the electrical properties of the electrode–solution interface – electrochemical properties of conducting polymers (Thompson et al., 2003, Cha et al., 2003, Wang et al., 1999), capacitance (Berney et al., 2000) or impedance (Souteyrand et al., 1997, Alfonta et al., 2001) – or using different covalently and non-covalently bound markers.",
        "Non-covalent markers (hybridization indicators) are electroactive compounds – metal complexes (Wang et al., 1997, Zhao and Ju, 2004, Nojima et al., 2003, Kara et al., 2002, Aoki and Umezawa, 2003) or organic compounds (Zhu et al., 2005, Teh et al., 2005, Marrazza et al., 1999, Hashimoto et al., 1998) – that interact in a different way with single strands (ss-) and double strands (ds-) of DNA.",
        "Covalently bound markers can be subdivided in two groups, electroactive and non-electroactive. Ferrocene and its derivatives (Nakayama et al., 2002, Xu et al., 2000, Xu et al., 2001), Pt(II) complex (Hernández-Santos et al., 2005), colloidal gold (Ozsoz et al., 2003, Cai et al., 2002), or tris(2,2′-bipyridyl)cobalt(III)-doped silica nanoparticles (Zhu et al., 2003) are used as electroactive markers, while as non-electroactive marker, the use of enzymes, linked directly to the DNA strand (Zhang et al., 2003, Zhang et al., 2004) or indirectly through biotin–avidin (Abad-Valle et al., 2005, Azek et al., 2000) or fluorescein–antifluorescein bridges (Hernández-Santos et al., 2004), has been extensively reported.",
        "As can be seen, metal complexes and metal nanoparticles have an important role in the development of electrochemical genosensors. In addition to the works cited above, where metal complexes are used as hybridization indicators or electroactive labels, examples of transition metal complexes, e.g. Ru(bpy)3\n2+, Os(bpy)3\n2+, mediating the electro-oxidation of nucleobases, have also been found in literature (Yang et al., 2002, Gore et al., 2003).",
        "In this work, it was used for first time sodium aurothiomalate as electroactive label in a DNA hybridization biosensor. This gold(I) complex has been previously used in our laboratory with success as electroactive label for immunosensor devices using glassy carbon electrodes (GCEs) as electrochemical transducers (de la Escosura-Muñiz et al., 2004a). The electrochemical detection in these immunosensors is based on the catalytic effect of ionic gold on silver electrodeposition (de la Escosura-Muñiz et al., 2004b).",
        "GCEs have been widely used in the construction of electrochemical DNA hybridization biosensors in combination with different oligonucleotides immobilization methods, such as (i) film entrapment using polymers (Wang et al., 1999, Xu et al., 2001, Cai et al., 2002, Zhu et al., 2003), (ii) covalent attachment on functionalized glassy carbon electrodes—polymers, carbon nanotubes, etc. (Zhao and Ju, 2004, Kara et al., 2002, Zhu et al., 2005, Teh et al., 2005, Piro et al., 2005, Cai et al., 2003) or (iii) by electrostatic adsorption on gold colloid modified surfaces (Lin et al., 2002).",
        "Direct adsorption of oligonucleotides onto pre-oxidized glassy carbon electrodes is used in this work, in combination with the electroactive label sodium aurothiomalate for the electrochemical detection of a SARS virus sequence, using a silver catalytic electrodeposition method.",
        "Cyclic voltammetric experiments were performed with an ECO CHEMIE Autolab PGSTAT 10 potentiostat interfaced to a Pentium 120 computer system and controlled by an Autolab GPES Version 4.6 for Windows 98.",
        "All measurements were carried out at room temperature in a 20 mL cell (protected from light) with a three-electrode configuration. Working glassy carbon electrodes (GCEs) were home made, using 3 mm diameter glassy carbon rods (Goodfellow, Spain), sealed into Teflon holders with a spurr resin for electron microscopy purchased from Sigma (Spain). Electrical contact was a brass rod welded to glassy carbon with a silver loaded conductive epoxy resin purchased from Circuit Works (USA). The renewal of the glassy carbon surface was achieved by polishing with 1.0 and 0.3 μm alpha-alumina on a microcloth polishing sheet of 8 in, followed by washing in an ultrasonic Selecta bath for 5 min.",
        "A platinum wire as counter electrode and an Ag/AgCl reference electrode were used.",
        "A Metrohm AG Herisau magnetic stirrer was used for the electrochemical pre-treatment of the electrode surface and the gold electrodeposition and oxidation.",
        "Synthetic 30-mer oligonucleotides were obtained from Eurogentec (Spain). The target sequence employed corresponds to a portion of the severe acute respiratory syndrome (SARS) virus, precisely the bases comprised between 29218 and 29247, both included. For selectivity studies, single-base and three-base-mismatch strands were also purchased. Mismatches are located in bases number 12 and 5, 15 and 26, respectively.Target (50 nmol): 5′-ACA-GAG-CCT-AAA-AAG-GAC-AAA-AAG-AAA-AAG-3′;Thiolated target (50 nmol): 5′-ACA-GAG-CCT-AAA-AAG-GAC-AAA-AAG-AAA-AAG-SH-3′;Single-base-mismatch target (49 nmol): 5′-ACA-GAG-CCT-AAC-AAG-GAC-AAA-AAG-AAA-AAG-3′;Three-base-mismatch target (184 nmol): 5′-ACA-GCG-CCT-AAA-AAC-GAC-AAA-AAG-AGA-AAG-3′;The probe is a complementary strand of the target that was also obtained from Eurogentec. Probe (50 nmol): 5′-CTT-TTT-CTT-TTT-GTC-CTT-TTT-AGG-CTC-TGT-3′.\n",
        "Oligonucleotide solutions were prepared in TE buffer, pH 8 (10 mM Tris–HCl buffer solution, 1 mM in EDTA). Aliquots were prepared and maintained at −20 °C.",
        "Working solutions of the oligonucleotide probe were made in 0.1 M Tris, pH 7.2 buffer, while all gold labeled oligonucleotide target strands were diluted in a 2 × SSC buffer (300 mM sodium chloride/30 mM sodium citrate), pH 7.2. These solutions were conserved at 4 °C. For selectivity studies, oligonucleotide strands were diluted in 2 × SCC buffer or in this buffer containing 50% of formamide. For the labeling procedure, target strands were diluted in an aqueous solution of 0.15 M NaCl with pH adjusted to 7.5 with 0.1 M NaOH.",
        "Sodium aurothiomalate was obtained from Aldrich (Spain). It was reconstituted in 0.15 M NaCl solution and protected from light. Dilutions of this stock solution were prepared daily in an unbuffered aqueous solution of 0.15 M NaCl, with pH adjusted to 7.5 with 0.1 M NaOH.",
        "Tris(hydroxymethyl)aminomethane (Tris) and formamide were purchased from Sigma (Spain). EDTA was obtained from Fluka (Spain).",
        "Silver nitrate was obtained from Sigma (Spain). Solutions were prepared using ultra-pure water and were protected from light.",
        "Analytical grade (Merck, Spain) NaCl, HCl, H2SO4, NH3, KCN, NaOH and sodium citrate were used. Their solutions were prepared with Millipore Milli-Q system, excepting KCN solutions, which were prepared with a 0.1 M NaOH solution.",
        "Ultra-pure water obtained with a Milli-Q plus 185 from Millipore Ibérica S.A. (Spain) was used for all solutions.",
        "Alumina powders of 1.0 and 0.3 μm grain sizes and microcloth polishing sheet of 8 in were purchased from Buehler (Germany).",
        "Dialysis procedures were carried out with Slide-A-Lizer dialysis cassettes, 3500 MWCO from Pierce (USA).",
        "The conjugation of aurothiomalate to single strands was carried out according to the procedure described in a previous work to label immunoglobulins (de la Escosura-Muñiz et al., 2004a). An aliquot of 50 μL of a 200 ng μL−1 target strand solution was mixed with an aliquot of 450 μL of a 390 ng μL−1 sodium aurothiomalate solution. The reaction was carried out at 37 °C for 24 h. After that, the conjugate (strand-Au) was purified by dialysis against 200 mL of 0.15 M NaCl, pH 7.5, unbuffered solution, for 48 h at room temperature.",
        "Thiolated and non-thiolated targets, single-base and three-base-mismatch complementary strands and a non-complementary strand were labeled following this procedure.",
        "The smoothed glassy carbon surfaces were pre-treated before each assay by an electrochemical pre-treatment in 0.1 M HCl. Working electrodes were immersed in a 0.1 M HCl stirred solution and a potential of +1.40 V was applied for 2 min. After that, the electrode surfaces were washed with 0.1 M Tris–HCl, pH 7.2, buffer.",
        "The probe immobilization was performed by physical adsorption, onto the inverted electrode, at room temperature. Five GCEs were used simultaneously and the adsorption was carried out depositing a 50 μL droplet of a 1 ng μL−1 oligonucleotide probe solution on each electrode surface, and left there for 60 min. Then, a washing step was performed with the hybridization buffer solution.",
        "After the probe immobilization and the washing step, the hybridization reaction took place for 30 min with a 30 μL droplet of the complementary gold labeled strand, 2 × SCC buffer pH 7.2 diluted, at room temperature. After that, the electrode surfaces were washed with ultra-pure water.",
        "The same procedure was performed to obtain the background signals, using a gold labeled non-complementary strand, and for selectivity studies with gold labeled single-base and three-base-mismatch complementary strands. For the selectivity studies, 2 × SCC buffer, pH 7.2, and this buffer containing 50% of formamide were used.",
        "The electrodes were immersed in a stirred 0.1 M HCl solution, and the gold, which was bound to the target strand, was electrodeposited on the electrode surfaces by applying a potential of −1.00 V for 5 min. After a washing step with ultra-pure water, the electrodes were immersed in a stirred 0.1 M H2SO4 solution and an oxidation step was carried out applying a potential of +1.40 V for 60 s. After that, the electrodes were rinsed with ultra-pure water and introduced in an unstirred 1.0 M NH3 solution containing silver nitrate at a fixed concentration (2.0 × 10−4\n M) and held at a deposition potential of −0.14 V for 60 s. Then, cyclic voltammograms were scanned from deposition potential to +0.30 V at a scan rate of 50 mV s−1, obtaining the analytical signal. Finally, in order to remove the gold from the electrode surfaces, the GCEs were immersed after each measurement in another cell containing a 0.1 M KCN stirred solution for 2 min in open circuit.",
        "This analytical procedure, based on the catalytic effect of electrodeposited gold on the electroreduction of silver ions (de la Escosura-Muñiz et al., 2004b) has been previously optimized, using bovine serum albumin (BSA) and a rabbit immunoglobulin G (RIgG) labeled with aurothiomalate (de la Escosura-Muñiz et al., 2004a). Briefly, this detection protocol involves a reduction step at −1.00 V in HCl 0.1 M for the gold label electrodeposition on the electrode surface, followed by an oxidation step in 0.1 M H2SO4 necessary to remove the hydrogen generated in the gold deposition step and to oxidize this gold. Then, by selecting an adequate potential (−0.14 V), the silver contained in the solution of AgNO3 2 × 10−4\n M–NH3 1.0 M is reduced to metallic silver only in presence of gold deposited on the electrode surface. Finally, an anodic scan is performed and the reduced silver is oxidized at +0.08 V, which constitutes the analytical signal.",
        "As the amount of silver electrodeposited at this potential is proportional to the gold deposited on the electrode surface, the silver stripping allows the determination of the gold labeled target that has hybridized with the probe. When a non-complementary gold labeled strand is assayed no hybridization takes place, so no gold is deposited on the electrode surface. Thus, no silver is reduced at the chosen deposition potential, and no signal is obtained.",
        "It is well known that sodium aurothiomalate binds to the albumin molecules mainly through the thiol group of the cysteine 34, by exchanging a proton (Moller-Pedersen, 1987). Based on this interaction between gold and thiol, our group has successfully used sodium autothiomalate for labeling different biological molecules, such as BSA and immunoglobulins, in order to design electrochemical metal immunoassays and immunosensors (de la Escosura-Muñiz et al., 2004a).",
        "Thus, this high affinity between gold and thiol group was applied in order to label a thiolated oligonucleotide. The product of the dialysis step was then evaluated by direct adsorption on the electrode surface, previously pre-treated at +1.40 V (versus Ag/AgCl) for 2 min in 0.1 M HCl stirred solution. A 50 μL droplet of a 1 ng μL−1 solution of the thiolated oligonucleotide, after the labeling process, was deposited on the electrode surface and left there for 30 min, before recording the analytical signal.",
        "To verify the gold(I)–oligonucleotide binding and to know if this interaction is through the thiol group, a solution of sodium aurothiomalate 390 ng μL−1 that was not mixed with any oligonucleotide, and a non-thiolated oligonucleotide with identical sequence, were subjected to the same procedure (reaction + dialysis + adsorption).",
        "The results obtained are showed in Fig. 2\n. As it can be seen in this figure, very similar analytical signals were obtained for both, the thiolated (Fig. 2a) and the non-thiolated oligonucleotide (Fig. 2b), while, as it could be expected, no analytical signal was obtained for the aurothiomalate solution assayed (Fig. 2c), because, as it is known by our previous works, sodium aurothiomalate is totally dialysed and anyway, it is not adsorbed on the electrode surface under these experimental conditions. The reproducibility of the signals showed in Fig. 2a and b was checked, using five different GCEs, obtaining a R.S.D. of 8% (n\n = 5, Ip average = 6.6 μA) and 7% (n\n = 5, Ip average = 6.4 μA), respectively.",
        "These results indicate that oligonucleotide is adsorbed on the GCE surface and it has been successfully labeled with sodium aurothiomalate.",
        "The DNA binding properties of sodium aurothiomalate and other gold(I) complexes have been studied by Blank and Dabrowiak (1984) using absorption and circular dichroism spectroscopies. This study concludes that sodium aurothiomalate does not bind to calf thymus DNA, while others gold(I) complexes which have an easily displaced ligand, e.g. Cl− or Br− are capable of interacting in a non-denaturing fashion with the guanine and cytosine residues of calf thymus DNA.",
        "To the best of our knowledge, this is the unique study of gold(I)–DNA complexes that includes sodium aurothiomalate. So, we only have evidence of its not interaction with calf thymus DNA, but we have no information about its binding to ssDNA or to DNA nucleosides.",
        "Therefore, the aurothiomalate–oligonucleotide binding mechanism is not clear, since not only the thiolated strand was labeled, being the interaction not through the thiol group, at least in its totality, as it could be expect. Besides, this fact allows us to label unmodified oligonucleotides which is economically advantageous.",
        "Physical adsorption is used in order to immobilize the oligonucleotide on the pre-oxidized GCE surface at room temperature. The influence of the probe strand concentration and adsorption time on the analytical signal were evaluated following the analytical procedure previously described, using in both cases 30 min of hybridization and 0.1 ng μL−1 gold labeled target solution.",
        "\nFig. 3\nshows the influence of the probe strand concentration on the analytical signal, using an adsorption time of 60 min. The peak current increases with the probe concentration up to 1 ng μL−1, where it reaches a plateau. So, 1 ng μL−1 was chosen for further studies.",
        "A very important fact obtained from this optimization assay is that non-specific adsorptions of the gold labeled target on the electrode surface are not observed for the concentration assayed (0.1 ng μL−1), since when no probe is immobilized, no analytical signal is recorded. The absence of non-specific adsorptions is a very important fact in the development of DNA hybridization sensors, since a blocking step is not needed, which simplifies the procedure, and the sensitivity of the assay is notably improved.",
        "Two factors could be responsible for this fact. One of them is that the gold labeled target concentration is not higher enough to adsorb on the electrode surface in 30 min. The other one is that, although the strand is adsorbed, the low amount of gold deposited on the electrode does not catalyze the silver reduction.",
        "Once the probe concentration was optimized, the effect of the probe adsorption time (from 5 to 90 min) on the analytical signal was evaluated. The obtained results (data not shown) show that the peak current increases upon raising the adsorption time up to 60 min. Beyond this point, the electrode response changes slightly with the adsorption time. Thus, an adsorption time of 60 min is chosen for further studies. Besides of the peak current increase observed with the adsorption time, a better reproducibility of the analytical signal is also obtained when the adsorption time is increased.",
        "Electrostatic adsorption of the oligonucleotide probe was also tested using different times (1–10 min) and potentials (+0.3 to +1.0 V) without success.",
        "After the optimization of the genosensor sensing phase, the hybridization reaction time (between 5 and 60 min) was studied using a 30 μL droplet of a 0.1 ng μL−1 solution of gold labeled target strand in 2 × SCC, pH 7.2, buffer solution. The analytical signal recorded increases with the reaction time, reaching a plateau for a hybridization time of 30 min (data not shown). So, a hybridization time of 30 min is chosen for further studies.",
        "The background signal was also evaluated using a 0.1 ng μL−1 solution of the gold labeled non-complementary strand. When this strand is assayed, no hybridization reaction is expected, so this gold labeled strand will not be captured by the probe immobilized on the electrode surface. As the analytical signal is based on the catalytic effect of electrodeposited gold on the silver electroreduction, when no gold is deposited on the electrode surface, no silver is reduced at the chosen deposition potential, and no signal is obtained.",
        "As can be seen in Fig. 4\n, no hybridization reaction takes place working with the gold labeled non-complementary strand, and a perfect discrimination between complementary (Fig. 4a) and non-complementary (Fig. 4b) strand is obtained under the experimental conditions optimized.",
        "Following the analytical procedure previously described, the higher the concentration of labeled target in solution, the higher is the amount of gold that is electrodeposited, and subsequently, an increase in the analytical signal is obtained.",
        "The peak current showed a good linear relationship with the concentration of gold labeled target prepared in 2 × SCC, pH 7.2, buffer solution, in the range from 10 to 200 pg μL−1, with a correlation coefficient of 0.9984, according to the following equation:Ip (μA)=0.049[target] (pg μL−1)+0.198 (n=5).\n",
        "The limit of detection (calculated as the concentration corresponding to three times the standard deviation of the estimate) of the target strand was 5.0 pg μL−1 (15 fmol in 30 μL).",
        "Background signals, using the gold labeled non-complementary strand, and non-specific adsorption of the gold labeled target were also evaluated. As it was expected in both cases, no analytical signals were obtained working with concentrations between 10 and 200 pg μL−1.",
        "The reproducibility of the analytical signal was checked. Using five different GCEs, a R.S.D. of 10% (n\n = 5, Ip average = 2.1 μA) was obtained for a 50 pg μL−1 solution of the complementary strand.",
        "Finally, it was also evaluated the selectivity of the DNA hybridization sensor, using different concentrations (10, 25, 50, 100 and 200 pg μL−1) of both single-base and three-base-mismatch gold labeled complementary strands prepared in 2 × SCC, pH 7.2, buffer solution. As it is shown in Fig. 5\n, the sensor allows us to discriminate between the complementary strand and the three-base-mismatch complementary strand. However, with the single-base mismatch complementary strand, the analytical signals recorded are similar than those obtained for the complementary strand.",
        "In order to discriminate a single-base-mismatch, other saline concentrations of the hybridization buffer were tested, but the discrimination between single-base-mismatch complementary strand and the target strand was not improved.",
        "More stringent experimental conditions were also tested, adding different concentrations of formamide to the 2 × SCC, pH 7.2, hybridization buffer. It is well known that this molecule makes the hybridization reaction more difficult by decreasing the melting point of DNA.",
        "A concentration of 50% of formamide was necessary to discriminate between single-base-mismatch complementary strand and the target strand. The results obtained for 10, 25, 50, 100 and 200 pg μL−1 strand concentrations are shown in Fig. 6\n. Working with this formamide concentration, the analytical signals obtained for the target strand decrease, while the analytical signals obtained for the three-base-mismatch complementary strand were approaching zero.",
        "Under these more stringent conditions, a new calibration curve was obtained. A linear relationship between peak current and concentration of oligonucleotide target was obtained for concentrations between 10 and 200 pg μL−1, with a correlation coefficient of 0.997, according to the following equation:Ip (μA)=0.016[target] (pg μL−1)+0.0925 (n=5).\n",
        "The limit of detection calculated as described above, was 10 pg μL−1 (30 fmol in 30 μL).",
        "As can be seen, the sensitivity (slope) of the assay decreases under these more stringent conditions, due to the minor peak current recorded for all concentrations of gold labeled target assayed. However, the same linear range from 10 to 200 pg μL−1 were obtained working with both non-stringent and stringent experimental conditions.",
        "The R.S.D. (50 pg μL−1 target strand) for five parallel experiments under these experimental conditions was 8.9% with a mean peak current of 0.84 μA.",
        "This work describes, for the first time, the use of sodium aurothiomalate as electroactive label in a DNA hybridization biosensor.",
        "About the interaction between this gold(I)complex and oligonucleotides, we only can conclude that the presence of a thiol group on the strand, which acts as a binding site for gold in other biological molecules, is not necessary to carry out an effective interaction.",
        "However, the hybridization of gold labeled target strand onto the surface of a probe modified GCE is performed successfully, demonstrating that DNA bases of the target are available for the hybridization after its labeling with the gold(I) complex.",
        "Although the sensitivity of the sensor is lower than that obtained with an enzymatic method on gold band electrodes (Abad-Valle et al., 2005), the analysis time is considerably shorter, since in the enzymatic method, a blocking step, a biotin–streptavidin interaction to link the enzyme to the duplex and an enzymatic reaction are needed before to obtain the analytical signal.",
        "Besides, the sensitivity of this genosensor is competitive with others reported that use Pt(II) complex – 20-mer oligonucleotide (Hernández-Santos et al., 2005) – or tris(2,2′-bipyridil)cobalt(III)-doped silica nanoparticle – 24-mer oligonucleotide (Zhu et al., 2003) – electroactive label; and with others gold nanoparticle-based hybridization assays – 19-mer oligonucleotide (Wang et al., 2001) – that use electrochemical detection.",
        "The reported genosensor is simple, economical and it is able to discriminate single-base mismatch on the target.",
        "Work is in progress trying to elucidate the binding mechanism between aurothiomalate and oligonucleotides and adapting the system to another carbon electrodes in order to improve the sensitivity."
      ],
      tags: {
        sciwingI2B2: {
          "0,10": "problem",
          "0,11": "problem",
          "0,12": "problem",
          "0,13": "problem",
          "0,14": "problem",
          "0,17": "problem",
          "0,24": "problem",
          "0,37": "problem",
          "0,40": "problem",
          "0,41": "problem",
          "0,43": "problem",
          "0,45": "problem",
          "0,46": "problem",
          "0,47": "problem",
          "0,48": "problem",
          "0,49": "problem",
          "0,51": "problem",
          "0,52": "problem",
          "0,53": "problem",
          "0,68": "problem",
          "0,71": "problem",
          "0,72": "problem",
          "0,77": "problem",
          "0,78": "problem",
          "0,87": "problem",
          "0,88": "problem",
          "0,90": "problem",
          "0,91": "problem",
          "0,107": "problem",
          "0,108": "problem",
          "0,109": "problem",
          "0,115": "problem",
          "0,116": "problem",
          "0,117": "problem",
          "0,121": "problem",
          "0,122": "problem",
          "0,123": "problem",
          "0,125": "problem",
          "0,128": "problem",
          "0,129": "problem",
          "1,0": "problem",
          "1,5": "problem",
          "1,6": "problem",
          "1,8": "problem",
          "1,9": "problem",
          "1,13": "problem",
          "1,14": "problem",
          "1,15": "problem",
          "1,16": "problem",
          "1,17": "problem",
          "1,18": "problem",
          "1,23": "problem",
          "1,24": "problem",
          "1,25": "problem",
          "1,28": "problem",
          "1,37": "problem",
          "1,45": "problem",
          "1,46": "problem",
          "1,51": "problem",
          "1,52": "problem",
          "1,54": "problem",
          "1,66": "treatment",
          "1,68": "treatment",
          "1,69": "treatment",
          "1,70": "treatment",
          "1,71": "treatment",
          "1,72": "treatment",
          "1,74": "treatment",
          "1,76": "treatment",
          "1,77": "treatment",
          "1,78": "treatment",
          "1,79": "treatment",
          "1,80": "treatment",
          "1,82": "treatment",
          "1,90": "problem",
          "1,91": "problem",
          "1,95": "problem",
          "1,96": "problem",
          "1,105": "problem",
          "1,106": "problem",
          "1,108": "test",
          "1,109": "test",
          "1,123": "problem",
          "1,124": "problem",
          "1,126": "problem",
          "1,127": "problem",
          "1,128": "problem",
          "1,131": "problem",
          "1,132": "problem",
          "1,133": "problem",
          "1,134": "problem",
          "1,135": "problem",
          "1,139": "problem",
          "1,140": "problem",
          "1,142": "problem",
          "1,143": "problem",
          "1,144": "problem",
          "1,146": "problem",
          "1,147": "problem",
          "1,149": "problem",
          "1,150": "problem",
          "1,151": "problem",
          "1,152": "problem",
          "1,157": "problem",
          "1,158": "problem",
          "1,159": "problem",
          "1,160": "problem",
          "1,162": "problem",
          "1,163": "problem",
          "1,165": "problem",
          "1,166": "problem",
          "1,167": "problem",
          "1,168": "problem",
          "1,173": "problem",
          "1,174": "problem",
          "1,175": "problem",
          "1,176": "problem",
          "1,184": "problem",
          "1,189": "problem",
          "1,190": "problem",
          "1,192": "problem",
          "1,193": "problem",
          "2,0": "problem",
          "2,1": "problem",
          "2,2": "problem",
          "2,4": "problem",
          "2,5": "problem",
          "2,6": "problem",
          "2,7": "problem",
          "2,31": "problem",
          "2,35": "test",
          "2,36": "test",
          "2,45": "problem",
          "2,70": "test",
          "2,71": "test",
          "2,73": "problem",
          "2,86": "problem",
          "2,87": "problem",
          "2,90": "test",
          "2,91": "test",
          "2,93": "problem",
          "2,94": "problem",
          "2,95": "problem",
          "2,96": "problem",
          "2,98": "problem",
          "2,99": "problem",
          "2,100": "problem",
          "2,101": "problem",
          "2,103": "problem",
          "2,105": "problem",
          "2,106": "problem",
          "2,107": "problem",
          "2,113": "problem",
          "2,114": "problem",
          "2,116": "problem",
          "2,117": "problem",
          "2,118": "problem",
          "2,121": "problem",
          "2,122": "problem",
          "2,127": "problem",
          "2,128": "problem",
          "2,136": "problem",
          "2,137": "problem",
          "2,138": "problem",
          "2,139": "problem",
          "2,145": "problem",
          "2,150": "problem",
          "2,155": "test",
          "2,156": "test",
          "2,167": "problem",
          "2,169": "problem",
          "2,170": "problem",
          "2,171": "problem",
          "2,172": "problem",
          "2,173": "problem",
          "2,174": "problem",
          "2,175": "problem",
          "2,176": "problem",
          "2,178": "problem",
          "2,179": "problem",
          "2,188": "problem",
          "2,189": "problem",
          "2,192": "problem",
          "2,193": "problem",
          "2,194": "problem",
          "2,195": "problem",
          "2,197": "problem",
          "2,199": "test",
          "2,200": "test",
          "2,201": "test",
          "2,202": "test",
          "2,209": "problem",
          "2,222": "problem",
          "2,223": "problem",
          "2,224": "problem",
          "2,229": "problem",
          "2,230": "problem",
          "2,236": "problem",
          "2,237": "problem",
          "3,3": "problem",
          "3,4": "problem",
          "3,5": "problem",
          "3,23": "problem",
          "3,24": "problem",
          "3,29": "problem",
          "3,30": "problem",
          "3,31": "problem",
          "3,37": "test",
          "3,38": "test",
          "3,48": "problem",
          "3,49": "problem",
          "3,50": "problem",
          "3,54": "problem",
          "3,55": "problem",
          "3,60": "problem",
          "3,61": "problem",
          "3,68": "problem",
          "3,69": "problem",
          "3,71": "problem",
          "3,72": "problem",
          "3,73": "problem",
          "3,77": "problem",
          "3,85": "problem",
          "3,86": "problem",
          "3,88": "problem",
          "3,89": "problem",
          "3,97": "problem",
          "3,98": "problem",
          "3,99": "problem",
          "3,100": "problem",
          "3,101": "problem",
          "3,102": "problem",
          "3,103": "problem",
          "3,104": "problem",
          "3,106": "problem",
          "3,116": "test",
          "3,117": "test",
          "3,122": "problem",
          "3,123": "problem",
          "3,127": "problem",
          "3,128": "problem",
          "3,133": "test",
          "3,141": "problem",
          "3,144": "problem",
          "3,148": "problem",
          "3,150": "problem",
          "3,151": "problem",
          "3,152": "problem",
          "3,153": "problem",
          "3,154": "problem",
          "3,156": "problem",
          "3,157": "problem",
          "4,0": "problem",
          "4,1": "problem",
          "4,2": "problem",
          "4,4": "problem",
          "4,5": "problem",
          "4,11": "problem",
          "4,13": "problem",
          "4,15": "problem",
          "4,16": "problem",
          "4,17": "problem",
          "4,23": "problem",
          "4,24": "problem",
          "4,28": "problem",
          "4,30": "problem",
          "4,31": "problem",
          "4,32": "problem",
          "4,33": "problem",
          "4,35": "problem",
          "4,36": "problem",
          "4,45": "problem",
          "4,46": "problem",
          "4,49": "problem",
          "4,50": "problem",
          "4,51": "problem",
          "4,52": "problem",
          "4,54": "problem",
          "4,56": "problem",
          "4,68": "problem",
          "4,69": "problem",
          "4,75": "problem",
          "4,80": "problem",
          "4,81": "problem",
          "4,82": "problem",
          "5,0": "treatment",
          "5,2": "problem",
          "5,4": "test",
          "5,5": "test",
          "5,6": "test",
          "5,11": "problem",
          "5,12": "problem",
          "5,16": "problem",
          "5,21": "treatment",
          "5,22": "treatment",
          "5,24": "problem",
          "5,25": "problem",
          "5,26": "problem",
          "5,31": "problem",
          "5,32": "problem",
          "5,36": "treatment",
          "5,37": "treatment",
          "5,38": "treatment",
          "5,43": "treatment",
          "5,44": "treatment",
          "5,57": "test",
          "6,0": "test",
          "6,1": "test",
          "6,7": "problem",
          "6,8": "problem",
          "6,15": "test",
          "6,17": "test",
          "6,18": "test",
          "6,22": "treatment",
          "6,23": "treatment",
          "6,25": "treatment",
          "6,26": "treatment",
          "6,31": "treatment",
          "6,32": "treatment",
          "6,33": "treatment",
          "6,34": "treatment",
          "6,35": "treatment",
          "6,36": "treatment",
          "7,0": "problem",
          "7,1": "problem",
          "7,2": "problem",
          "7,3": "problem",
          "7,5": "problem",
          "7,6": "problem",
          "7,7": "problem",
          "7,8": "problem",
          "7,16": "problem",
          "7,19": "problem",
          "7,22": "problem",
          "7,24": "problem",
          "7,28": "problem",
          "7,29": "problem",
          "7,31": "problem",
          "7,32": "problem",
          "7,33": "problem",
          "7,36": "treatment",
          "7,37": "treatment",
          "7,39": "test",
          "7,40": "test",
          "7,45": "treatment",
          "7,46": "treatment",
          "7,47": "test",
          "7,48": "test",
          "7,51": "test",
          "7,53": "problem",
          "7,54": "problem",
          "7,55": "problem",
          "7,58": "treatment",
          "7,61": "problem",
          "7,63": "test",
          "7,64": "test",
          "7,66": "problem",
          "7,68": "problem",
          "7,69": "problem",
          "7,72": "treatment",
          "7,77": "test",
          "7,78": "test",
          "7,83": "test",
          "7,88": "test",
          "7,92": "test",
          "7,93": "test",
          "7,94": "test",
          "8,1": "test",
          "8,2": "test",
          "8,4": "test",
          "8,13": "test",
          "8,29": "test",
          "8,30": "test",
          "8,36": "treatment",
          "8,52": "test",
          "8,53": "test",
          "8,58": "treatment",
          "8,67": "treatment",
          "8,68": "treatment",
          "8,70": "test",
          "8,71": "test",
          "8,73": "treatment",
          "8,77": "test",
          "8,78": "test",
          "8,91": "test",
          "8,92": "test",
          "8,103": "test",
          "8,109": "test",
          "8,110": "test",
          "8,112": "test",
          "8,115": "problem",
          "8,117": "test",
          "8,118": "test",
          "8,120": "test",
          "9,6": "problem",
          "9,7": "problem",
          "9,17": "problem",
          "9,21": "test",
          "9,22": "test",
          "9,28": "problem",
          "9,30": "problem",
          "9,36": "test",
          "9,37": "test",
          "9,40": "treatment",
          "9,41": "treatment",
          "9,43": "problem",
          "9,44": "problem",
          "9,45": "problem",
          "9,46": "problem",
          "9,49": "problem",
          "9,50": "problem",
          "9,51": "problem",
          "9,57": "problem",
          "9,62": "treatment",
          "9,71": "problem",
          "9,78": "problem",
          "9,80": "problem",
          "9,81": "problem",
          "9,83": "treatment",
          "9,91": "treatment",
          "9,92": "treatment",
          "9,125": "treatment",
          "9,128": "problem",
          "9,129": "problem",
          "9,130": "problem",
          "9,131": "problem",
          "9,132": "problem",
          "9,133": "problem",
          "9,135": "problem",
          "9,136": "problem",
          "9,138": "treatment",
          "9,151": "test",
          "9,163": "treatment",
          "9,164": "treatment",
          "9,166": "problem",
          "9,168": "test",
          "9,169": "test",
          "9,171": "test",
          "9,172": "test",
          "9,173": "test",
          "9,180": "problem",
          "9,181": "problem",
          "9,184": "test",
          "9,185": "test",
          "9,187": "problem",
          "9,188": "problem",
          "9,190": "problem",
          "9,191": "problem",
          "9,192": "problem",
          "9,198": "problem",
          "9,200": "problem",
          "9,201": "problem",
          "9,202": "problem",
          "9,203": "problem",
          "9,207": "treatment",
          "9,218": "treatment",
          "9,220": "problem",
          "9,221": "problem",
          "9,222": "problem",
          "9,223": "problem",
          "9,229": "treatment",
          "9,230": "treatment",
          "9,232": "treatment",
          "9,233": "treatment",
          "9,239": "test",
          "9,240": "test",
          "9,241": "test",
          "9,242": "test",
          "9,250": "treatment",
          "9,251": "treatment",
          "9,252": "treatment",
          "9,264": "test",
          "9,265": "test",
          "9,304": "problem",
          "9,305": "problem",
          "9,311": "treatment",
          "9,312": "treatment",
          "9,313": "treatment",
          "9,314": "treatment",
          "9,315": "test",
          "9,316": "test",
          "9,329": "problem",
          "9,331": "problem",
          "9,332": "problem",
          "9,336": "treatment",
          "9,340": "problem",
          "9,341": "problem",
          "9,344": "test",
          "9,345": "test",
          "9,346": "test",
          "9,350": "treatment",
          "9,352": "test",
          "9,353": "test",
          "9,359": "problem",
          "9,360": "problem",
          "9,362": "problem",
          "9,363": "problem",
          "9,364": "problem",
          "9,366": "test",
          "9,367": "test",
          "9,368": "test",
          "9,370": "test",
          "9,372": "problem",
          "9,374": "treatment",
          "9,375": "treatment",
          "9,376": "treatment",
          "9,377": "test",
          "9,378": "test",
          "9,389": "treatment",
          "9,390": "treatment",
          "9,391": "treatment",
          "9,400": "treatment",
          "9,416": "problem",
          "9,425": "test",
          "9,426": "test",
          "9,434": "treatment",
          "9,436": "treatment",
          "9,448": "treatment",
          "9,449": "treatment",
          "9,450": "test",
          "9,451": "test",
          "9,466": "test",
          "9,467": "test",
          "9,468": "test",
          "9,472": "problem",
          "9,474": "problem",
          "10,13": "problem",
          "10,14": "problem",
          "10,18": "problem",
          "10,19": "problem",
          "10,21": "test",
          "10,22": "test",
          "10,23": "test",
          "10,25": "test",
          "10,27": "test",
          "10,28": "test",
          "10,29": "test",
          "10,30": "test",
          "10,31": "test",
          "10,38": "problem",
          "10,40": "problem",
          "10,46": "problem",
          "10,47": "problem",
          "10,55": "problem",
          "10,56": "problem",
          "10,60": "problem",
          "10,61": "problem",
          "10,68": "problem",
          "10,69": "problem",
          "10,70": "problem",
          "10,71": "problem",
          "10,72": "problem",
          "10,74": "test",
          "10,75": "test",
          "10,85": "test",
          "10,86": "test",
          "10,89": "test",
          "10,90": "test",
          "10,91": "test",
          "10,92": "test",
          "10,97": "test",
          "10,98": "test",
          "10,100": "test",
          "10,101": "test",
          "10,102": "test",
          "10,103": "test",
          "10,105": "test",
          "10,108": "test",
          "10,109": "test",
          "10,113": "test",
          "10,114": "test",
          "10,115": "test",
          "10,116": "test",
          "10,117": "test",
          "10,121": "problem",
          "10,122": "problem",
          "10,126": "test",
          "10,128": "problem",
          "10,129": "problem",
          "10,131": "problem",
          "10,132": "problem",
          "10,133": "problem",
          "10,135": "test",
          "10,136": "test",
          "10,138": "test",
          "10,139": "test",
          "10,143": "test",
          "10,144": "test",
          "10,145": "test",
          "10,149": "test",
          "10,150": "test",
          "10,155": "test",
          "10,156": "test",
          "10,157": "test",
          "10,163": "treatment",
          "10,164": "treatment",
          "10,165": "treatment",
          "11,6": "problem",
          "11,7": "problem",
          "11,23": "problem",
          "11,25": "problem",
          "11,26": "problem",
          "11,32": "test",
          "11,33": "test",
          "11,39": "test",
          "11,40": "test",
          "11,41": "test",
          "11,42": "test",
          "11,45": "problem",
          "11,46": "problem",
          "11,47": "problem",
          "11,49": "problem",
          "11,57": "test",
          "11,58": "test",
          "11,59": "test",
          "11,67": "test",
          "11,68": "test",
          "11,69": "test",
          "11,74": "treatment",
          "11,75": "treatment",
          "11,76": "treatment",
          "11,77": "treatment",
          "11,78": "treatment",
          "11,79": "treatment",
          "11,80": "treatment",
          "11,81": "treatment",
          "11,82": "treatment",
          "11,84": "treatment",
          "11,85": "treatment",
          "11,86": "treatment",
          "11,88": "treatment",
          "11,89": "treatment",
          "11,92": "treatment",
          "11,93": "treatment",
          "11,94": "treatment",
          "11,95": "treatment",
          "11,96": "treatment",
          "11,97": "treatment",
          "11,101": "test",
          "11,102": "test",
          "11,105": "problem",
          "11,106": "problem",
          "11,107": "problem",
          "11,108": "problem",
          "11,109": "problem",
          "11,111": "problem",
          "11,112": "problem",
          "11,120": "treatment",
          "11,121": "treatment",
          "11,122": "treatment",
          "11,125": "problem",
          "11,127": "problem",
          "11,128": "problem",
          "11,130": "test",
          "11,131": "test",
          "11,136": "test",
          "11,137": "test",
          "11,139": "test",
          "11,144": "test",
          "11,145": "test",
          "11,146": "test",
          "11,147": "test",
          "11,148": "test",
          "11,150": "test",
          "11,151": "test",
          "11,152": "test",
          "11,153": "test",
          "11,154": "test",
          "11,157": "test",
          "11,158": "test",
          "11,159": "test",
          "11,161": "test",
          "11,162": "test",
          "11,163": "test",
          "11,164": "test",
          "11,165": "test",
          "11,167": "problem",
          "11,168": "problem",
          "11,175": "test",
          "11,176": "test",
          "11,177": "test",
          "12,6": "problem",
          "12,7": "problem",
          "12,9": "problem",
          "12,10": "problem",
          "12,12": "test",
          "12,14": "test",
          "12,15": "test",
          "12,16": "test",
          "12,18": "problem",
          "12,19": "problem",
          "12,26": "treatment",
          "12,27": "treatment",
          "13,0": "test",
          "13,1": "test",
          "13,3": "problem",
          "13,19": "treatment",
          "13,20": "treatment",
          "13,26": "treatment",
          "13,27": "treatment",
          "13,37": "problem",
          "13,38": "problem",
          "13,42": "problem",
          "13,45": "test",
          "13,46": "test",
          "13,47": "test",
          "14,0": "test",
          "14,1": "test",
          "14,2": "test",
          "14,3": "test",
          "14,4": "test",
          "14,5": "test",
          "14,6": "test",
          "14,7": "test",
          "14,14": "test",
          "14,15": "test",
          "14,23": "test",
          "14,24": "test",
          "14,25": "test",
          "14,29": "test",
          "14,30": "test",
          "14,31": "test",
          "14,32": "test",
          "14,33": "test",
          "14,43": "problem",
          "14,44": "problem",
          "14,49": "test",
          "14,50": "test",
          "14,53": "test",
          "14,54": "test",
          "14,55": "test",
          "14,56": "test",
          "14,65": "test",
          "14,68": "test",
          "14,69": "test",
          "14,70": "test",
          "14,78": "test",
          "14,80": "problem",
          "14,84": "test",
          "14,85": "test",
          "14,86": "test",
          "14,87": "test",
          "14,88": "test",
          "14,90": "test",
          "14,91": "test",
          "14,92": "test",
          "14,97": "problem",
          "14,99": "test",
          "14,100": "test",
          "14,102": "problem",
          "14,103": "problem",
          "14,104": "problem",
          "14,106": "test",
          "14,107": "test",
          "14,114": "test",
          "14,115": "test",
          "14,118": "test",
          "14,119": "test",
          "14,131": "test",
          "14,132": "test",
          "14,135": "problem",
          "14,136": "problem",
          "14,138": "problem",
          "14,139": "problem",
          "14,143": "test",
          "14,144": "test",
          "14,149": "test",
          "14,152": "problem",
          "14,155": "problem",
          "14,158": "treatment",
          "14,159": "treatment",
          "14,160": "treatment",
          "14,162": "treatment",
          "14,163": "treatment",
          "14,164": "treatment",
          "14,165": "treatment",
          "14,168": "problem",
          "14,169": "problem",
          "14,170": "problem",
          "14,175": "problem",
          "14,176": "problem",
          "14,177": "problem",
          "14,178": "problem",
          "14,183": "problem",
          "14,186": "problem",
          "14,187": "problem",
          "14,192": "test",
          "14,194": "problem",
          "14,195": "problem",
          "14,196": "problem",
          "14,199": "problem",
          "14,200": "problem",
          "15,0": "test",
          "15,1": "test",
          "15,2": "test",
          "15,3": "test",
          "15,10": "test",
          "15,11": "test",
          "15,12": "treatment",
          "15,17": "test",
          "15,18": "test",
          "15,20": "test",
          "15,21": "test",
          "15,22": "test",
          "15,23": "test",
          "15,24": "test",
          "15,25": "test",
          "15,28": "test",
          "15,29": "test",
          "15,30": "test",
          "15,33": "treatment",
          "15,36": "treatment",
          "15,37": "treatment",
          "15,38": "treatment",
          "15,39": "treatment",
          "15,40": "treatment",
          "15,41": "treatment",
          "15,44": "treatment",
          "15,45": "treatment",
          "15,46": "treatment",
          "15,47": "treatment",
          "15,48": "treatment",
          "15,49": "treatment",
          "15,51": "test",
          "15,52": "test",
          "15,53": "test",
          "15,57": "treatment",
          "15,58": "treatment",
          "15,62": "test",
          "15,63": "test",
          "15,65": "treatment",
          "15,66": "treatment",
          "15,67": "treatment",
          "15,68": "treatment",
          "15,70": "treatment",
          "15,71": "treatment",
          "15,72": "treatment",
          "15,73": "treatment",
          "16,0": "test",
          "16,1": "test",
          "16,13": "problem",
          "16,14": "problem",
          "16,15": "problem",
          "16,18": "problem",
          "16,19": "problem",
          "16,21": "test",
          "16,22": "test",
          "16,23": "test",
          "16,24": "test",
          "16,25": "test",
          "16,26": "test",
          "16,28": "test",
          "16,29": "test",
          "16,30": "test",
          "16,33": "test",
          "16,34": "test",
          "16,35": "test",
          "16,36": "test",
          "16,37": "test",
          "16,40": "problem",
          "16,41": "problem",
          "16,42": "problem",
          "16,49": "test",
          "16,50": "test",
          "16,51": "test",
          "16,52": "test",
          "16,54": "problem",
          "16,56": "test",
          "16,58": "test",
          "16,59": "test",
          "16,68": "test",
          "16,69": "test",
          "16,70": "test",
          "16,72": "test",
          "16,73": "test",
          "16,86": "problem",
          "16,87": "problem",
          "16,88": "problem",
          "16,90": "problem",
          "16,91": "treatment",
          "16,92": "treatment",
          "16,93": "treatment",
          "16,94": "treatment",
          "16,101": "test",
          "16,103": "problem",
          "16,106": "problem",
          "16,108": "problem",
          "16,115": "test",
          "16,116": "test",
          "16,117": "test",
          "16,118": "test",
          "16,120": "test",
          "16,121": "test",
          "16,122": "test",
          "16,124": "test",
          "16,125": "test",
          "16,126": "test",
          "16,127": "test",
          "16,129": "test",
          "16,130": "test",
          "16,134": "test",
          "16,135": "test",
          "16,139": "problem",
          "16,140": "problem",
          "16,141": "problem",
          "16,142": "problem",
          "16,143": "problem",
          "16,144": "problem",
          "16,147": "test",
          "16,148": "test",
          "16,149": "test",
          "16,150": "test",
          "16,152": "test",
          "16,158": "test",
          "16,159": "test",
          "16,160": "test",
          "16,165": "problem",
          "16,174": "treatment",
          "16,175": "treatment",
          "16,176": "treatment",
          "16,179": "test",
          "16,180": "test",
          "16,181": "test",
          "16,184": "test",
          "16,185": "test",
          "16,188": "treatment",
          "16,189": "treatment",
          "16,194": "treatment",
          "16,195": "treatment",
          "16,196": "treatment",
          "16,197": "treatment",
          "16,198": "treatment",
          "16,199": "treatment",
          "16,210": "test",
          "16,211": "test",
          "17,0": "test",
          "17,1": "test",
          "17,2": "test",
          "17,3": "test",
          "17,9": "test",
          "17,10": "test",
          "17,14": "test",
          "17,15": "test",
          "17,16": "test",
          "17,17": "test",
          "17,18": "test",
          "17,19": "test",
          "17,20": "test",
          "17,21": "test",
          "17,22": "test",
          "17,25": "test",
          "17,26": "test",
          "17,28": "test",
          "17,29": "test",
          "17,32": "problem",
          "17,33": "problem",
          "17,35": "test",
          "17,44": "test",
          "17,45": "test",
          "17,46": "test",
          "17,48": "problem",
          "17,49": "problem",
          "17,52": "problem",
          "17,54": "problem",
          "17,55": "problem",
          "17,61": "problem",
          "17,62": "problem",
          "17,67": "test",
          "17,69": "test",
          "17,70": "test",
          "17,72": "test",
          "17,73": "test",
          "17,75": "test",
          "17,76": "test",
          "17,77": "test",
          "17,81": "test",
          "17,82": "test",
          "17,86": "test",
          "17,87": "test",
          "17,90": "test",
          "17,91": "test",
          "17,92": "test",
          "17,94": "test",
          "17,95": "test",
          "17,97": "treatment",
          "18,1": "test",
          "18,2": "test",
          "18,5": "problem",
          "18,8": "problem",
          "18,23": "problem",
          "18,26": "test",
          "18,27": "test",
          "18,28": "test",
          "18,29": "test",
          "18,30": "test",
          "18,33": "test",
          "18,34": "test",
          "18,35": "test",
          "18,37": "test",
          "18,38": "test",
          "18,41": "test",
          "18,42": "test",
          "18,43": "test",
          "18,44": "test",
          "18,45": "test",
          "18,46": "test",
          "18,47": "test",
          "18,52": "problem",
          "18,53": "problem",
          "18,55": "test",
          "18,56": "test",
          "18,58": "test",
          "18,63": "problem",
          "18,64": "problem",
          "18,65": "problem",
          "18,69": "test",
          "18,70": "test",
          "18,74": "test",
          "18,75": "test",
          "18,76": "test",
          "18,77": "test",
          "18,78": "test",
          "18,85": "problem",
          "18,87": "problem",
          "18,89": "problem",
          "18,91": "test",
          "18,92": "test",
          "18,95": "problem",
          "18,96": "problem",
          "18,105": "problem",
          "18,112": "problem",
          "18,113": "problem",
          "18,114": "problem",
          "18,115": "problem",
          "19,1": "test",
          "19,4": "problem",
          "19,11": "test",
          "19,12": "test",
          "19,13": "test",
          "19,16": "problem",
          "19,18": "test",
          "19,19": "test",
          "19,20": "test",
          "19,21": "test",
          "19,22": "test",
          "19,26": "test",
          "19,27": "test",
          "19,29": "test",
          "19,30": "test",
          "19,32": "test",
          "19,33": "test",
          "19,35": "test",
          "19,36": "test",
          "19,38": "test",
          "19,39": "test",
          "19,40": "test",
          "19,41": "test",
          "19,43": "test",
          "19,45": "test",
          "19,46": "test",
          "19,47": "test",
          "19,48": "test",
          "19,52": "problem",
          "19,53": "problem",
          "19,54": "problem",
          "19,56": "test",
          "19,57": "test",
          "19,58": "test",
          "19,59": "test",
          "19,60": "test",
          "19,62": "test",
          "19,63": "test",
          "19,65": "test",
          "19,66": "test",
          "19,70": "test",
          "19,71": "test",
          "19,72": "test",
          "19,73": "test",
          "19,74": "test",
          "19,75": "test",
          "19,76": "test",
          "19,77": "test",
          "19,78": "test",
          "19,79": "test",
          "19,85": "problem",
          "19,86": "problem",
          "19,88": "test",
          "19,89": "test",
          "19,90": "test",
          "19,92": "problem",
          "19,93": "problem",
          "19,95": "problem",
          "19,96": "problem",
          "19,97": "problem",
          "19,98": "problem",
          "19,104": "problem",
          "19,105": "problem",
          "19,106": "problem",
          "19,107": "problem",
          "19,111": "problem",
          "19,113": "test",
          "19,114": "test",
          "19,115": "test",
          "19,118": "test",
          "19,119": "test",
          "19,120": "test",
          "19,122": "problem",
          "19,133": "problem",
          "19,134": "problem",
          "19,135": "problem",
          "19,136": "problem",
          "19,137": "problem",
          "19,138": "problem",
          "19,142": "test",
          "19,143": "test",
          "19,147": "test",
          "19,151": "test",
          "20,0": "test",
          "20,1": "test",
          "20,2": "test",
          "20,7": "problem",
          "20,8": "problem",
          "20,19": "test",
          "20,20": "test",
          "20,22": "test",
          "20,23": "test",
          "20,24": "test",
          "20,29": "problem",
          "20,32": "problem",
          "20,33": "problem",
          "20,34": "problem",
          "20,35": "problem",
          "20,40": "problem",
          "20,42": "problem",
          "20,43": "problem",
          "20,47": "treatment",
          "20,48": "treatment",
          "20,49": "treatment",
          "21,4": "problem",
          "21,26": "treatment",
          "21,28": "problem",
          "21,29": "problem",
          "21,30": "problem",
          "21,31": "problem",
          "21,33": "problem",
          "21,34": "problem",
          "21,35": "problem",
          "21,38": "test",
          "21,47": "treatment",
          "21,51": "problem",
          "21,52": "problem",
          "21,54": "problem",
          "21,55": "problem",
          "22,6": "problem",
          "22,7": "problem",
          "22,10": "treatment",
          "22,11": "treatment",
          "22,20": "treatment",
          "22,28": "treatment",
          "22,29": "treatment",
          "22,30": "treatment",
          "22,38": "treatment",
          "22,39": "treatment",
          "22,40": "treatment",
          "22,51": "problem",
          "22,61": "problem",
          "22,62": "problem",
          "22,63": "problem",
          "22,64": "problem",
          "22,66": "problem",
          "22,67": "problem",
          "22,68": "problem",
          "22,69": "problem",
          "22,70": "problem",
          "22,72": "problem",
          "22,73": "problem",
          "22,77": "treatment",
          "22,78": "treatment",
          "22,82": "treatment",
          "22,83": "treatment",
          "22,86": "problem",
          "22,87": "problem",
          "22,93": "treatment",
          "22,94": "treatment",
          "22,109": "treatment",
          "22,110": "treatment",
          "22,122": "test",
          "22,123": "test",
          "22,134": "treatment",
          "22,135": "treatment",
          "22,136": "treatment",
          "22,144": "treatment",
          "22,145": "treatment",
          "22,155": "problem",
          "22,156": "problem",
          "22,157": "problem",
          "22,159": "problem",
          "22,160": "problem",
          "22,161": "test",
          "22,162": "test",
          "22,164": "treatment",
          "22,165": "treatment",
          "22,166": "treatment",
          "22,168": "treatment",
          "22,169": "treatment",
          "22,170": "treatment",
          "22,174": "problem",
          "22,176": "problem",
          "22,178": "problem",
          "22,185": "treatment",
          "22,186": "treatment",
          "22,191": "problem",
          "22,192": "problem",
          "22,193": "problem",
          "22,194": "problem",
          "22,196": "treatment",
          "22,197": "treatment",
          "22,198": "treatment",
          "22,199": "treatment",
          "22,204": "problem",
          "22,205": "problem",
          "22,207": "problem",
          "22,208": "problem",
          "22,209": "problem",
          "22,214": "problem",
          "22,215": "problem",
          "22,232": "problem",
          "22,234": "problem",
          "22,235": "problem",
          "22,236": "problem",
          "22,243": "treatment",
          "22,244": "treatment",
          "22,245": "treatment",
          "22,246": "treatment"
        }
      }
    },
    url:
      "https://www.ncbi.nlm.nih.gov/pubmed/16762539/; https://api.elsevier.com/content/article/pii/S0956566306002296; https://www.sciencedirect.com/science/article/pii/S0956566306002296"
  },
  {
    answer: {
      score: "96.63428994184085",
      sents: [
        "In addition to direct government contributions, innovative finance mechanisms have been successful in raising funds for vaccines in the past and should be used to fund the development of COVID-19 vaccines.",
        "8, 9 The International Finance Facility for Immunisation (IFFIm) raises funds with vaccine bonds, which turn long-term contributions by donors into available cash.",
        "8 IFFIm was created to support Gavi, the Vaccine Alliance, but could be used to finance CEPI's COVID-19 vaccine efforts.",
        "With advanced market commitments, donors make funding commitments to vaccine manufacturers and, in exchange, companies sign a legally binding commitment to provide the vaccines at a price affordable to low-income and middle-income countries.",
        "Gavi's board expressed support for the use of Gavi's IFFIm and advanced market commitments to improve COVID-19 vaccine development and access.",
        "10 The need for COVID-19 vaccines is global, although the need is differentially distributed within populations.",
        "Vaccines would likely be prioritised for health-care workers and people at greatest risk of severe illness and death.",
        "High-income countries must not monopolise the global supply of COVID-19 vaccines.",
        "This risk is real: during the 2009 influenza A/H1N1 pandemic, rich countries negotiated large advance orders for the vaccine, crowding out poor countries.",
        "11 Such an outcome would result in a suboptimal allocation of an initially scarce resource."
      ]
    },

    paper_id: "d0dafaa2700d9fa0289b620ce87992f9cf393ae1",
    doi: "10.1016/s0140-6736(20)30763-7",
    doc_date: "2020-03-31",
    url: "https://doi.org/10.1016/j.bbrc.2020.03.047",
    title:
      "Spike protein recognition of mammalian ACE2 predicts the host range and an optimized ACE2 for SARS-CoV-2 infection",
    authors: ["Junwen.Luan", "Yue.Lu", "Xiaolu.Jin", "Leiliang.Zhang"],
    summary: "",
    abstract: {
      text: [
        "a b s t r a c t SARS-CoV-2 causes the recent global COVID-19 public health emergency",
        " ACE2 is the receptor for both SARS-CoV-2 and SARS-CoV",
        " To predict the potential host range of SARS-CoV-2, we analyzed the key residues of ACE2 for recognizing S protein",
        " We found that most of the selected mammals including pets (dog and cat), pangolin and Circetidae mammals remained the most of key residues for association with S protein from SARS-CoV and SARS-CoV-2",
        " The interaction interface between cat/dog/pangolin/Chinese hamster ACE2 and SARS-CoV/SARS-CoV-2 S protein was simulated through homology modeling",
        " We identified that N82 in ACE2 showed a closer contact with SARS-CoV-2 S protein than M82 in human ACE2",
        " Our finding will provide important insights into the host range of SARS-CoV-2 and a new strategy to design an optimized ACE2 for SARS-CoV-2 infection",
        "\nPlease cite this article as: J",
        " Luan et al",
        ", Spike protein recognition of mammalian ACE2 predicts the host range and an optimized ACE2 for SARS-CoV-2 infection, Biochemical and Biophysical Research Communications, https://doi"
      ],
      tags: {
        sciwing: [
          "background",
          "finding",
          "method",
          "finding",
          "finding",
          "finding",
          "finding",
          "other",
          "other",
          "finding"
        ]
      }
    },
    bodyText: {
      text: [
        "Corona Virus Disease 2019 (COVID- 19) , which was reported from Wuhan city, Hubei province of China, has caused over 78,000 human infections and more than 2700 deaths (as of February 25, 2020) [1, 2] . Severe Acute Respiratory Syndrome Corona Virus 2 (SARS-CoV-2) was identified as the pathogen of COVID- 19 [1e3] . After SARS-CoV and MERS-CoV, SARS-CoV-2 has become the third coronavirus that causes severe respiratory disease and human death [4, 5] .",
        "Belonging to the subgenus sarbecvirus of Coronaviridae, both SARS-CoV-2 and SARS-CoV are human SARS-related coronavirus (SARSr-CoV). Its genome is a single-stranded RNA composed of about 30 kb nucleotides. SARS-CoV-2 encodes at least four major structural proteins, namely spike protein (S), membrane protein (M), envelope protein (E), and nucleocapsid protein (N) [6] . S protein, which is a type I glycoprotein, protrudes from the surface of the virus and can contact the host cell earlier. S protein has attracted great attention because of its function in receptor binding.",
        "Angiotensin-converting enzyme 2 (ACE2) binds to the receptorbinding motif (RBM) in the receptor-binding domain (RBD) of SARS-CoV and functions as a receptor for SARS-CoV [7, 8] . ACE2 is widely distributed in heart, liver, testis, kidney, intestine and other tissues. It has the physiological functions of regulating heart and kidney function and controlling blood pressure [9] . Recently, it has been found that human ACE2 promoted the entry of SARS-CoV-2 into the cells [3, 10] . RBD domain of SARS-CoV-2 interacts with human ACE2. Thus, ACE2 is defined as the receptor for SARS-CoV-2.",
        "The specificity of the interaction between virus and receptor determines the host tropism and host range. The origin of SARS-CoV-2 is presumed to be bat [3] . However, the intermediate host is not clear, and some studies suggest that pangolin is involved in the evolution of SARS-CoV-2 [11, 12] . It is not clear which mammals are involved in the evolution of SARS-CoV-2 and which animals may be infected by SARS-CoV-2. By sequence alignment of key amino acids binding to RBD in ACE2, the interaction between RBD of SARS-CoV-2/SARS-CoV and mammalian ACE2 was predicted. Based on the potential interaction between S protein and mammalian ACE2, it was speculated that SARS-CoV-2/SARS-CoV preserved the ability to infect many mammals including cat, dog, pangolin and Chinese hamster. From the structure stimulation, we identified N82 in ACE2 show closer contact with F486 of SARS-CoV-2 S protein compared with M82 of ACE2.",
        "The S protein sequence of SARS-CoV-2 is YP_009724390.1, and the S protein sequence of SARS-CoV is NP_828851.1. RBM of SARS-CoV is from 424 to 494. RBM of SARS-CoV-2 is from 437 to 508.",
        "A total of 42 mammalian ACE2 protein sequences were selected from the wild animal protection lists of Hubei Province and Jiangxi Province, primates, bats, dog, and cat. These sequences are as follows: hACE2: Homo sapiens (BAB40370. Cavia porcellus (XP_023417808.1), CgACE2: Cricetulus griseus (A0A061HZ66). Based on the known key sites in SARS-CoV S protein interacting with human ACE2, we analyzed whether these sites were conserved on ACE2 from wild mammals and domestic pets. Phylogenetic and molecular evolutionary analyses of ACE2 were conducted using MEGA version X [13] . Phylogenetic tree was generated with Jones-Taylor-Thornton (JTT) evolutionary model using Maximum Likelihood method.",
        "The interaction interfaces of SARSr-CoV S and ACE2 from cat/ dog/pangolin/Chinese hamster were simulated by Chimera software Ver 1.14 [14] . The simulation were based on the structures of hACE2 with SARS-CoV S RBD (PDB: 2AJF) [7] and hACE2 with SARS-CoV-2 S RBD (PDB: 6LZG).",
        "Human ACE2 is the receptor for both SARS-CoV and SARS-CoV-2. According to the literature [15] , the key amino acids (AAs) in the S protein of SARS-CoV interacting with human ACE2 are Y442, L472, N479, D480, T487, Y491 [7, 15] . We compared the RBM in S protein of SARS-CoV-2 with that of SARS-CoV (Fig. 1A) . These amino acids corresponding to SARS-CoV-2 are L455, F486, Q493, S494, N501 and Y505 (Fig. 1A) . Although five of six key AAs in SARS-CoV-2 are changed compared with SARS-CoV, the overall structure of interfaces of ACE2-RBM are similar (Fig. 1B) .",
        "The key AAs in hACE2 for interacting with RBM are K31, E35, D38, M82 and K353 [7, 15] . Among them, K31 and K353 in hACE2 are most critical residues for RBM recognition. Because the overall structure of interfaces of ACE2-RBM in SARS-CoV-2 and SARS-CoV are similar, we analyzed the key RBD recognizing AAs of ACE2 protein from selected mammals, as shown in Table 1 . We predicted that the mammals whose ACE2 could bind to the S protein of SARS- Next, we constructed a phylogenetic tree for mammalian ACE2 proteins. The ACE2 protein sequences of 42 mammalian animals were compared by ClustalW method of MEGA-X software. Then the JTT model of maximum likelihood method was used to construct ACE2 phylogenetic tree. As shown in Fig. 2 , species that cannot bind to S protein are marked in red, and species that can bind to S protein are marked in green. No correlation between genetic distance and the interaction of ACE2/S was found. Some Pets including dog (Canis lupus familiaris) and cat (Felis catus) potentially recognize S protein, indicating the importance to monitor the pets for SARS-CoV-2 infection. We found that four members of Circetidae including Mesocricetus auratus, Phodopus campbelli, Ictidomys tridecemlineatus, and Cricetulus griseus remained the key residues for association with S protein from SARS-CoV and SARS-CoV-2, though two members of Muridae (Rattus norvegicus, Mus musculus) could not bind to S protein. This founding suggested that Circetidae mammals could be developed as SARSr-CoV small animal models.",
        "We noticed that ACE2 from dog, cat, pangolin and Chinese hamster potentially associated with S protein (Table 1 ). Next, we predicted the structure of cat/dog/pangolin/Chinese hamster ACE2 with SARSr-CoV RBD. The structure of protein complex between RBD region of S protein of SARS-CoV and human ACE2 has been resolved (PDB: 2AJF) [7] . Recently, the structure of SARS-CoV-2 RBD with human ACE2 was also determined [16, 17] . We used Chimera software to display homologous model, and obtained the interaction complex structure of RBD region of SARSr-CoV (SARS-CoV-2 and SARS-CoV) and cat/dog/pangolin/Chinese hamster ACE2. Overall, the RBM structures of S protein of SARS-CoV-2 and SARS-CoV are similar (Fig. 3) . Interaction interface of SARSr-CoV RBD and cat/dog/pangolin/Chinese hamster ACE2 confirmed the potential interaction between SARS-CoV-2 and cat/dog/pangolin/ Chinese hamster ACE2, indicating that these ACE2 could support SARS-CoV-2 entry. The AA in 82 position of human ACE2 is M82, while the corresponding AA in cat, dog, pangolin and Chinese hamster ACE2 is T82, T81, N82 and N82, respectively. The distance between F486 of SARS-CoV-2 S protein and the corresponding AA in ACE2 is 3.753Å for human (Fig. 1A) , 2.695Å for dog (Fig. 3A) , 3.753Å for cat (Fig. 3B ), 1.621Å for pangolin (Fig. 3C) , and 2.024Å for Chinese hamster (Fig. 3D) , respectively. We concluded that N82 in ACE2 showed closer contact with F486 of SARS-CoV-2 S protein than M82 of ACE2.",
        "The host tropism of zoonotic coronavirus is hybrid, and it is important to determine the natural host and host range of coronavirus. In the past two decades, SARS-CoV, MERS-CoV and SARS-CoV-2 have caused serious outbreaks of human infectious diseases. All the three human coronaviruses originated from bats, but the intermediate hosts were different. SARS-CoV is believed to come from the Paguma larvata [18] , and the intermediate host of MERS-CoV is Camelus dromedaries [19] . The new coronavirus SARS-CoV-2 has recently caused a serious epidemic in China, but its intermediate hosts are not clear.",
        "S protein of SARS-CoV-2 interacts with human ACE2, which promotes the entry of SARS-CoV-2, indicating that human ACE2 is the receptor of SARS-CoV-2 [3] . ACE2 contains at least five key amino acids critical for binding S protein of SARSr-CoV [15] . Based on these five amino acids, we analyzed the corresponding amino Table 1 Prediction of the RBD binding capacity of mammalian ACE2.",
        "AA position  matched AA  binding capacity   31  35  38  82  353 hACE2",
        "Phylogenetic tree of mammalian ACE2 proteins. ACE2 sequences from a total of 42 mammals were analyzed by MEGA-X and the phylogenetic tree was constructed with JTT evolutionary model using Maximum Likelihood method. The red represents the species whose ACE2 cannot bind to S protein, and the green is the species whose ACE2 associate with S protein. (For interpretation of the references to colour in this figure legend, the reader is referred to the Web version of this article.)",
        "acids of different mammals to determine which mammalian ACE2 could interact with S protein of human SARSr-CoV. By analyzing the protein sequence of mammalian ACE2, we found that the ACE2 of Camelus dromedarius, Procyon lotor, Rhinolophus ferrumequinum, Rattus norvegicus, Mus musculus, Ornithorhynchus anatinus, Loxodonta africana, Erinaceus europaeus, Nyctereutes procyonoides, Suricata suricatta, Dipodomys ordii, and Cavia porcellus lose the capability to associate with S protein (Table 1) . These mammals could be ruled out from the potential host list for SARS-CoV-2. We found that S protein may bind to ACE2 from some wild mammals, which suggests that we should investigate whether these animals may be intermediate hosts for SARS-CoV-2. It has been reported that the RBM region in S protein of pangolin coronavirus is similar to that of S protein of SARS-CoV-2 [11, 12] , which may be involved in the recombination of SARS-CoV-2. We identified that N82 of pangolin ACE2 showed closer contact with RBD than human ACE2 (Fig. 3C) , indicating that pangolin ACE2 might show better affinity to SARS-CoV-2. This finding further supports the hypothesis that pangolin is involved in SARS-CoV-2 evolution. In current study, only a limited list of wild mammals is covered. In the future, we should select more mammals for study. Although no SARS-CoV-2 has been found in domestic cats and dogs, cat/dog ACE2 may bind to S protein of SARS-CoV-2. In the future, we should pay attention to monitoring whether domestic cats and dogs could be infected by SARS-CoV-2. Animal model is an important tool in the study of infectious diseases. ACE2 of mice cannot interact with SARS-CoV-2, so it cannot be used as animal model of SARS-CoV-2 directly. Some studies have generated mice transfected with human ACE2 as the models to study SARS-CoV [20] , and these mice can also be used as animal models for SARS-CoV-2 infection. Interestingly, we identified that N82 in ACE2 is closer to RBD than M82 (Fig. 3C and D) , indicating a novel strategy to design an optimized ACE2 for SARS-CoV-2 infection. We speculated that small peptide based on N82 of ACE2 might show higher affinity to SARS-CoV-2 RBD. We proposed that if M82 in human ACE2 was mutated to N82, the modified human ACE2 will enhance SARS-CoV-2 infection. In the future, those ideas will be tested in cell culture and animal model. We noticed that the ACE2 proteins from Circetidae (Mesocricetus auratus, Phodopus campbelli, Ictidomys tridecemlineatus, and Cricetulus griseus) are capable to recognize RBD. Mesocricetus auratus (golden hamster) and Cricetulus griseus (Chinese hamster) are experimental animals and our finding indicates the possibility to develop small animal models for SARS-CoV-2 infection using Chinese hamster and golden hamster.",
        "The authors declare that there are no conflicts of interest."
      ],
      section_header: {
        original: [
          "intro",
          "intro",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "conclusion",
          "conclusion",
          "conclusion",
          "conclusion"
        ],
        generic: [
          "intro",
          "intro",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "main",
          "conclusion",
          "conclusion",
          "conclusion",
          "conclusion"
        ]
      },
      tags: {
        sciwingI2B2: {
          "1,1": "treatment",
          "1,3": "test",
          "3,3": "problem",
          "5,1": "treatment",
          "5,3": "test",
          "4,3": "problem"
        }
      }
    },
    similar: [
      {
        paper_id: "paper id test 1",
        doi: "doi test",
        doc_date: "",
        title:
          "Biophysical characterization of the SARS-CoV-2 spike protein binding with the ACE2 receptor and implications for infectivity",
        authors: ["author test"],
        summary:
          "We computationally contrasted the binding interactions between human ACE2 and coronavirus spike protein receptor binding domain (RBD) of the 2002 epidemic-causing SARS-CoV-1, SARS-CoV-2, and bat coronavirus RaTG13 using the Rosetta energy function.\nWe find that the RBD of the spike protein of SARS-CoV-2 is highly optimized to achieve very strong binding with human ACE2 (hACE2) which is consistent with its enhanced infectivity.\nNotably, the SARS-CoV-2 RBD out-competes the angiotensin 2 receptor type I (ATR1) which is the native binding partner of ACE2 by 35% in terms of the calculated binding affinity.\nBy contrasting the spike protein SARS-CoV-2 Rosetta binding energy with ACE2 of different livestock and pet species we find strongest binding with bat ACE2 followed by human, feline, equine, canine and finally chicken.",
        abstract: {
          text: ["abstract text test"],
          tags: {
            sciwing: ["tag 1"]
          }
        },
        bodyText: {
          text: ["bodytext text test", "paragraph 2"],
          tags: {
            "0": {
              tagPerPara: {
                "0": "Label1",
                "1": "Label2"
              }
            },
            "1": {
              tagPerPara: {
                "0": "Label3",
                "1": "Label4"
              }
            }
          }
        }
      }
    ]
  }
];
